<?xml version="1.0" encoding="UTF-8"?>
<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Strict//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-strict.dtd">
<html xmlns="http://www.w3.org/1999/xhtml">
<head>
    <title>PYTHON API TEST RESULT</title>
    <meta name="generator" content="HTMLTestRunner 0.8.2.2"/>
    <meta http-equiv="Content-Type" content="text/html; charset=UTF-8"/>
    <link href="http://libs.baidu.com/bootstrap/3.0.3/css/bootstrap.min.css" rel="stylesheet">
    <script src="http://libs.baidu.com/jquery/2.0.0/jquery.min.js"></script>
    <script src="http://libs.baidu.com/bootstrap/3.0.3/js/bootstrap.min.js"></script>
    
<style type="text/css" media="screen">
body        { font-family: Microsoft YaHei,Tahoma,arial,helvetica,sans-serif;padding: 20px; font-size: 120%; }
table       { font-size: 100%; }
/* -- heading ---------------------------------------------------------------------- */
.heading {
    margin-top: 0ex;
    margin-bottom: 1ex;
}
.heading .description {
    margin-top: 4ex;
    margin-bottom: 6ex;
}
/* -- report ------------------------------------------------------------------------ */
#total_row  { font-weight: bold; }
.passCase   { color: #5cb85c; }
.failCase   { color: #d9534f; font-weight: bold; }
.errorCase  { color: #f0ad4e; font-weight: bold; }
.hiddenRow  { display: none; }
.testcase   { margin-left: 2em; }
</style>

</head>
<body >
<script language="javascript" type="text/javascript">
output_list = Array();
/*level 调整增加只显示通过用例的分类 --Findyou
0:Summary //all hiddenRow
1:Failed  //pt hiddenRow, ft none
2:Pass    //pt none, ft hiddenRow
3:All     //pt none, ft none
*/
function showCase(level) {
    trs = document.getElementsByTagName("tr");
    for (var i = 0; i < trs.length; i++) {
        tr = trs[i];
        id = tr.id;
        if (id.substr(0,2) == 'ft') {
            if (level == 2 || level == 0 ) {
                tr.className = 'hiddenRow';
            }
            else {
                tr.className = '';
            }
        }
        if (id.substr(0,2) == 'pt') {
            if (level < 2) {
                tr.className = 'hiddenRow';
            }
            else {
                tr.className = '';
            }
        }
    }
    //加入【详细】切换文字变化 --Findyou
    detail_class=document.getElementsByClassName('detail');
	//console.log(detail_class.length)
	if (level == 3) {
		for (var i = 0; i < detail_class.length; i++){
			detail_class[i].innerHTML="收起"
		}
	}
	else{
			for (var i = 0; i < detail_class.length; i++){
			detail_class[i].innerHTML="详细"
		}
	}
}
function showClassDetail(cid, count) {
    var id_list = Array(count);
    var toHide = 1;
    for (var i = 0; i < count; i++) {
        //ID修改 点 为 下划线 -Findyou
        tid0 = 't' + cid.substr(1) + '_' + (i+1);
        tid = 'f' + tid0;
        tr = document.getElementById(tid);
        if (!tr) {
            tid = 'p' + tid0;
            tr = document.getElementById(tid);
        }
        id_list[i] = tid;
        if (tr.className) {
            toHide = 0;
        }
    }
    for (var i = 0; i < count; i++) {
        tid = id_list[i];
        //修改点击无法收起的BUG，加入【详细】切换文字变化 --Findyou
        if (toHide) {
            document.getElementById(tid).className = 'hiddenRow';
            document.getElementById(cid).innerText = "详细"
        }
        else {
            document.getElementById(tid).className = '';
            document.getElementById(cid).innerText = "收起"
        }
    }
}
function html_escape(s) {
    s = s.replace(/&/g,'&amp;');
    s = s.replace(/</g,'&lt;');
    s = s.replace(/>/g,'&gt;');
    return s;
}
</script>
<div class='heading'>
<h1 style="font-family: Microsoft YaHei">PYTHON API TEST RESULT</h1>
<p class='attribute'><strong>测试人员 : </strong> mai</p>
<p class='attribute'><strong>开始时间 : </strong> 2019-05-07 20:54:39</p>
<p class='attribute'><strong>合计耗时 : </strong> 0:00:07.940138</p>
<p class='attribute'><strong>测试结果 : </strong> 共 98，通过 21，失败 33，错误 44，通过率= 21.43%</p>

<p class='description'>前程貸</p>
</div>


<p id='show_detail_line'>
<a class="btn btn-primary" href='javascript:showCase(0)'>概要{ 21.43% }</a>
<a class="btn btn-danger" href='javascript:showCase(1)'>失败{ 33 }</a>
<a class="btn btn-success" href='javascript:showCase(2)'>通过{ 21 }</a>
<a class="btn btn-info" href='javascript:showCase(3)'>所有{ 98 }</a>
</p>
<table id='result_table' class="table table-condensed table-bordered table-hover">
<colgroup>
<col align='left' />
<col align='right' />
<col align='right' />
<col align='right' />
<col align='right' />
<col align='right' />
</colgroup>
<tr id='header_row' class="text-center success" style="font-weight: bold;font-size: 16px;">
    <td>用例集/测试用例</td>
    <td>总计</td>
    <td>通过</td>
    <td>失败</td>
    <td>错误</td>
    <td>详细</td>
</tr>

<tr class='errorClass warning'>
    <td>test_add.TestAdd: 測試添加標的</td>
    <td class="text-center">43</td>
    <td class="text-center">5</td>
    <td class="text-center">0</td>
    <td class="text-center">38</td>
    <td class="text-center"><a href="javascript:showClassDetail('c1',43)" class="detail" id='c1'>详细</a></td>
</tr>

<tr id='pt1_1' class='hiddenRow'>
    <td class='passCase'><div class='testcase'>test_add_01</div></td>
    <td colspan='5' align='center'><span class="label label-success success">通过</span></td>
</tr>

<tr id='pt1_2' class='hiddenRow'>
    <td class='passCase'><div class='testcase'>test_add_02</div></td>
    <td colspan='5' align='center'><span class="label label-success success">通过</span></td>
</tr>

<tr id='pt1_3' class='hiddenRow'>
    <td class='passCase'><div class='testcase'>test_add_03</div></td>
    <td colspan='5' align='center'><span class="label label-success success">通过</span></td>
</tr>

<tr id='pt1_4' class='hiddenRow'>
    <td class='passCase'><div class='testcase'>test_add_04</div></td>
    <td colspan='5' align='center'><span class="label label-success success">通过</span></td>
</tr>

<tr id='pt1_5' class='hiddenRow'>
    <td class='passCase'><div class='testcase'>test_add_05</div></td>
    <td colspan='5' align='center'><span class="label label-success success">通过</span></td>
</tr>

<tr id='ft1_6' class='none'>
    <td class='errorCase'><div class='testcase'>test_add_06</div></td>
    <td colspan='5' align='center'>
    <!--默认收起错误信息 -Findyou
    <button id='btn_ft1_6' type="button"  class="btn btn-danger btn-xs collapsed" data-toggle="collapse" data-target='#div_ft1_6'>错误</button>
    <div id='div_ft1_6' class="collapse">  -->
    <!-- 默认展开错误信息 -Findyou -->
    <button id='btn_ft1_6' type="button"  class="btn btn-danger btn-xs" data-toggle="collapse" data-target='#div_ft1_6'>错误</button>
    <div id='div_ft1_6' class="collapse in" align="left">
    <pre>
    
ft1_6: Traceback (most recent call last):
  File "C:\Python37\lib\site-packages\ddt.py", line 145, in wrapper
    return func(self, *args, **kwargs)
  File "D:\PycharmProjects\test2-24\cookies0409\UseExcel\testcases\test_add.py", line 35, in test_add
    case.data = replace(case.data)  # 若存在寫在配置文件中的賬號和密碼、項目名，就必須通過replace函數去拿到正則表達式匹配的值
  File "D:\PycharmProjects\test2-24\cookies0409\UseExcel\common\context.py", line 19, in replace
    da = str(da['id'])                   #從數據庫取出來的id是int
TypeError: 'NoneType' object is not subscriptable


    </pre>
    </div>
    </td>
</tr>

<tr id='ft1_7' class='none'>
    <td class='errorCase'><div class='testcase'>test_add_07</div></td>
    <td colspan='5' align='center'>
    <!--默认收起错误信息 -Findyou
    <button id='btn_ft1_7' type="button"  class="btn btn-danger btn-xs collapsed" data-toggle="collapse" data-target='#div_ft1_7'>错误</button>
    <div id='div_ft1_7' class="collapse">  -->
    <!-- 默认展开错误信息 -Findyou -->
    <button id='btn_ft1_7' type="button"  class="btn btn-danger btn-xs" data-toggle="collapse" data-target='#div_ft1_7'>错误</button>
    <div id='div_ft1_7' class="collapse in" align="left">
    <pre>
    
ft1_7: Traceback (most recent call last):
  File "C:\Python37\lib\site-packages\ddt.py", line 145, in wrapper
    return func(self, *args, **kwargs)
  File "D:\PycharmProjects\test2-24\cookies0409\UseExcel\testcases\test_add.py", line 35, in test_add
    case.data = replace(case.data)  # 若存在寫在配置文件中的賬號和密碼、項目名，就必須通過replace函數去拿到正則表達式匹配的值
  File "D:\PycharmProjects\test2-24\cookies0409\UseExcel\common\context.py", line 19, in replace
    da = str(da['id'])                   #從數據庫取出來的id是int
TypeError: 'NoneType' object is not subscriptable


    </pre>
    </div>
    </td>
</tr>

<tr id='ft1_8' class='none'>
    <td class='errorCase'><div class='testcase'>test_add_08</div></td>
    <td colspan='5' align='center'>
    <!--默认收起错误信息 -Findyou
    <button id='btn_ft1_8' type="button"  class="btn btn-danger btn-xs collapsed" data-toggle="collapse" data-target='#div_ft1_8'>错误</button>
    <div id='div_ft1_8' class="collapse">  -->
    <!-- 默认展开错误信息 -Findyou -->
    <button id='btn_ft1_8' type="button"  class="btn btn-danger btn-xs" data-toggle="collapse" data-target='#div_ft1_8'>错误</button>
    <div id='div_ft1_8' class="collapse in" align="left">
    <pre>
    
ft1_8: Traceback (most recent call last):
  File "C:\Python37\lib\site-packages\ddt.py", line 145, in wrapper
    return func(self, *args, **kwargs)
  File "D:\PycharmProjects\test2-24\cookies0409\UseExcel\testcases\test_add.py", line 35, in test_add
    case.data = replace(case.data)  # 若存在寫在配置文件中的賬號和密碼、項目名，就必須通過replace函數去拿到正則表達式匹配的值
  File "D:\PycharmProjects\test2-24\cookies0409\UseExcel\common\context.py", line 19, in replace
    da = str(da['id'])                   #從數據庫取出來的id是int
TypeError: 'NoneType' object is not subscriptable


    </pre>
    </div>
    </td>
</tr>

<tr id='ft1_9' class='none'>
    <td class='errorCase'><div class='testcase'>test_add_09</div></td>
    <td colspan='5' align='center'>
    <!--默认收起错误信息 -Findyou
    <button id='btn_ft1_9' type="button"  class="btn btn-danger btn-xs collapsed" data-toggle="collapse" data-target='#div_ft1_9'>错误</button>
    <div id='div_ft1_9' class="collapse">  -->
    <!-- 默认展开错误信息 -Findyou -->
    <button id='btn_ft1_9' type="button"  class="btn btn-danger btn-xs" data-toggle="collapse" data-target='#div_ft1_9'>错误</button>
    <div id='div_ft1_9' class="collapse in" align="left">
    <pre>
    
ft1_9: Traceback (most recent call last):
  File "C:\Python37\lib\site-packages\ddt.py", line 145, in wrapper
    return func(self, *args, **kwargs)
  File "D:\PycharmProjects\test2-24\cookies0409\UseExcel\testcases\test_add.py", line 35, in test_add
    case.data = replace(case.data)  # 若存在寫在配置文件中的賬號和密碼、項目名，就必須通過replace函數去拿到正則表達式匹配的值
  File "D:\PycharmProjects\test2-24\cookies0409\UseExcel\common\context.py", line 19, in replace
    da = str(da['id'])                   #從數據庫取出來的id是int
TypeError: 'NoneType' object is not subscriptable


    </pre>
    </div>
    </td>
</tr>

<tr id='ft1_10' class='none'>
    <td class='errorCase'><div class='testcase'>test_add_10</div></td>
    <td colspan='5' align='center'>
    <!--默认收起错误信息 -Findyou
    <button id='btn_ft1_10' type="button"  class="btn btn-danger btn-xs collapsed" data-toggle="collapse" data-target='#div_ft1_10'>错误</button>
    <div id='div_ft1_10' class="collapse">  -->
    <!-- 默认展开错误信息 -Findyou -->
    <button id='btn_ft1_10' type="button"  class="btn btn-danger btn-xs" data-toggle="collapse" data-target='#div_ft1_10'>错误</button>
    <div id='div_ft1_10' class="collapse in" align="left">
    <pre>
    
ft1_10: Traceback (most recent call last):
  File "C:\Python37\lib\site-packages\ddt.py", line 145, in wrapper
    return func(self, *args, **kwargs)
  File "D:\PycharmProjects\test2-24\cookies0409\UseExcel\testcases\test_add.py", line 35, in test_add
    case.data = replace(case.data)  # 若存在寫在配置文件中的賬號和密碼、項目名，就必須通過replace函數去拿到正則表達式匹配的值
  File "D:\PycharmProjects\test2-24\cookies0409\UseExcel\common\context.py", line 19, in replace
    da = str(da['id'])                   #從數據庫取出來的id是int
TypeError: 'NoneType' object is not subscriptable


    </pre>
    </div>
    </td>
</tr>

<tr id='ft1_11' class='none'>
    <td class='errorCase'><div class='testcase'>test_add_11</div></td>
    <td colspan='5' align='center'>
    <!--默认收起错误信息 -Findyou
    <button id='btn_ft1_11' type="button"  class="btn btn-danger btn-xs collapsed" data-toggle="collapse" data-target='#div_ft1_11'>错误</button>
    <div id='div_ft1_11' class="collapse">  -->
    <!-- 默认展开错误信息 -Findyou -->
    <button id='btn_ft1_11' type="button"  class="btn btn-danger btn-xs" data-toggle="collapse" data-target='#div_ft1_11'>错误</button>
    <div id='div_ft1_11' class="collapse in" align="left">
    <pre>
    
ft1_11: Traceback (most recent call last):
  File "C:\Python37\lib\site-packages\ddt.py", line 145, in wrapper
    return func(self, *args, **kwargs)
  File "D:\PycharmProjects\test2-24\cookies0409\UseExcel\testcases\test_add.py", line 35, in test_add
    case.data = replace(case.data)  # 若存在寫在配置文件中的賬號和密碼、項目名，就必須通過replace函數去拿到正則表達式匹配的值
  File "D:\PycharmProjects\test2-24\cookies0409\UseExcel\common\context.py", line 19, in replace
    da = str(da['id'])                   #從數據庫取出來的id是int
TypeError: 'NoneType' object is not subscriptable


    </pre>
    </div>
    </td>
</tr>

<tr id='ft1_12' class='none'>
    <td class='errorCase'><div class='testcase'>test_add_12</div></td>
    <td colspan='5' align='center'>
    <!--默认收起错误信息 -Findyou
    <button id='btn_ft1_12' type="button"  class="btn btn-danger btn-xs collapsed" data-toggle="collapse" data-target='#div_ft1_12'>错误</button>
    <div id='div_ft1_12' class="collapse">  -->
    <!-- 默认展开错误信息 -Findyou -->
    <button id='btn_ft1_12' type="button"  class="btn btn-danger btn-xs" data-toggle="collapse" data-target='#div_ft1_12'>错误</button>
    <div id='div_ft1_12' class="collapse in" align="left">
    <pre>
    
ft1_12: Traceback (most recent call last):
  File "C:\Python37\lib\site-packages\ddt.py", line 145, in wrapper
    return func(self, *args, **kwargs)
  File "D:\PycharmProjects\test2-24\cookies0409\UseExcel\testcases\test_add.py", line 35, in test_add
    case.data = replace(case.data)  # 若存在寫在配置文件中的賬號和密碼、項目名，就必須通過replace函數去拿到正則表達式匹配的值
  File "D:\PycharmProjects\test2-24\cookies0409\UseExcel\common\context.py", line 19, in replace
    da = str(da['id'])                   #從數據庫取出來的id是int
TypeError: 'NoneType' object is not subscriptable


    </pre>
    </div>
    </td>
</tr>

<tr id='ft1_13' class='none'>
    <td class='errorCase'><div class='testcase'>test_add_13</div></td>
    <td colspan='5' align='center'>
    <!--默认收起错误信息 -Findyou
    <button id='btn_ft1_13' type="button"  class="btn btn-danger btn-xs collapsed" data-toggle="collapse" data-target='#div_ft1_13'>错误</button>
    <div id='div_ft1_13' class="collapse">  -->
    <!-- 默认展开错误信息 -Findyou -->
    <button id='btn_ft1_13' type="button"  class="btn btn-danger btn-xs" data-toggle="collapse" data-target='#div_ft1_13'>错误</button>
    <div id='div_ft1_13' class="collapse in" align="left">
    <pre>
    
ft1_13: Traceback (most recent call last):
  File "C:\Python37\lib\site-packages\ddt.py", line 145, in wrapper
    return func(self, *args, **kwargs)
  File "D:\PycharmProjects\test2-24\cookies0409\UseExcel\testcases\test_add.py", line 35, in test_add
    case.data = replace(case.data)  # 若存在寫在配置文件中的賬號和密碼、項目名，就必須通過replace函數去拿到正則表達式匹配的值
  File "D:\PycharmProjects\test2-24\cookies0409\UseExcel\common\context.py", line 19, in replace
    da = str(da['id'])                   #從數據庫取出來的id是int
TypeError: 'NoneType' object is not subscriptable


    </pre>
    </div>
    </td>
</tr>

<tr id='ft1_14' class='none'>
    <td class='errorCase'><div class='testcase'>test_add_14</div></td>
    <td colspan='5' align='center'>
    <!--默认收起错误信息 -Findyou
    <button id='btn_ft1_14' type="button"  class="btn btn-danger btn-xs collapsed" data-toggle="collapse" data-target='#div_ft1_14'>错误</button>
    <div id='div_ft1_14' class="collapse">  -->
    <!-- 默认展开错误信息 -Findyou -->
    <button id='btn_ft1_14' type="button"  class="btn btn-danger btn-xs" data-toggle="collapse" data-target='#div_ft1_14'>错误</button>
    <div id='div_ft1_14' class="collapse in" align="left">
    <pre>
    
ft1_14: Traceback (most recent call last):
  File "C:\Python37\lib\site-packages\ddt.py", line 145, in wrapper
    return func(self, *args, **kwargs)
  File "D:\PycharmProjects\test2-24\cookies0409\UseExcel\testcases\test_add.py", line 35, in test_add
    case.data = replace(case.data)  # 若存在寫在配置文件中的賬號和密碼、項目名，就必須通過replace函數去拿到正則表達式匹配的值
  File "D:\PycharmProjects\test2-24\cookies0409\UseExcel\common\context.py", line 19, in replace
    da = str(da['id'])                   #從數據庫取出來的id是int
TypeError: 'NoneType' object is not subscriptable


    </pre>
    </div>
    </td>
</tr>

<tr id='ft1_15' class='none'>
    <td class='errorCase'><div class='testcase'>test_add_15</div></td>
    <td colspan='5' align='center'>
    <!--默认收起错误信息 -Findyou
    <button id='btn_ft1_15' type="button"  class="btn btn-danger btn-xs collapsed" data-toggle="collapse" data-target='#div_ft1_15'>错误</button>
    <div id='div_ft1_15' class="collapse">  -->
    <!-- 默认展开错误信息 -Findyou -->
    <button id='btn_ft1_15' type="button"  class="btn btn-danger btn-xs" data-toggle="collapse" data-target='#div_ft1_15'>错误</button>
    <div id='div_ft1_15' class="collapse in" align="left">
    <pre>
    
ft1_15: Traceback (most recent call last):
  File "C:\Python37\lib\site-packages\ddt.py", line 145, in wrapper
    return func(self, *args, **kwargs)
  File "D:\PycharmProjects\test2-24\cookies0409\UseExcel\testcases\test_add.py", line 35, in test_add
    case.data = replace(case.data)  # 若存在寫在配置文件中的賬號和密碼、項目名，就必須通過replace函數去拿到正則表達式匹配的值
  File "D:\PycharmProjects\test2-24\cookies0409\UseExcel\common\context.py", line 19, in replace
    da = str(da['id'])                   #從數據庫取出來的id是int
TypeError: 'NoneType' object is not subscriptable


    </pre>
    </div>
    </td>
</tr>

<tr id='ft1_16' class='none'>
    <td class='errorCase'><div class='testcase'>test_add_16</div></td>
    <td colspan='5' align='center'>
    <!--默认收起错误信息 -Findyou
    <button id='btn_ft1_16' type="button"  class="btn btn-danger btn-xs collapsed" data-toggle="collapse" data-target='#div_ft1_16'>错误</button>
    <div id='div_ft1_16' class="collapse">  -->
    <!-- 默认展开错误信息 -Findyou -->
    <button id='btn_ft1_16' type="button"  class="btn btn-danger btn-xs" data-toggle="collapse" data-target='#div_ft1_16'>错误</button>
    <div id='div_ft1_16' class="collapse in" align="left">
    <pre>
    
ft1_16: Traceback (most recent call last):
  File "C:\Python37\lib\site-packages\ddt.py", line 145, in wrapper
    return func(self, *args, **kwargs)
  File "D:\PycharmProjects\test2-24\cookies0409\UseExcel\testcases\test_add.py", line 35, in test_add
    case.data = replace(case.data)  # 若存在寫在配置文件中的賬號和密碼、項目名，就必須通過replace函數去拿到正則表達式匹配的值
  File "D:\PycharmProjects\test2-24\cookies0409\UseExcel\common\context.py", line 19, in replace
    da = str(da['id'])                   #從數據庫取出來的id是int
TypeError: 'NoneType' object is not subscriptable


    </pre>
    </div>
    </td>
</tr>

<tr id='ft1_17' class='none'>
    <td class='errorCase'><div class='testcase'>test_add_17</div></td>
    <td colspan='5' align='center'>
    <!--默认收起错误信息 -Findyou
    <button id='btn_ft1_17' type="button"  class="btn btn-danger btn-xs collapsed" data-toggle="collapse" data-target='#div_ft1_17'>错误</button>
    <div id='div_ft1_17' class="collapse">  -->
    <!-- 默认展开错误信息 -Findyou -->
    <button id='btn_ft1_17' type="button"  class="btn btn-danger btn-xs" data-toggle="collapse" data-target='#div_ft1_17'>错误</button>
    <div id='div_ft1_17' class="collapse in" align="left">
    <pre>
    
ft1_17: Traceback (most recent call last):
  File "C:\Python37\lib\site-packages\ddt.py", line 145, in wrapper
    return func(self, *args, **kwargs)
  File "D:\PycharmProjects\test2-24\cookies0409\UseExcel\testcases\test_add.py", line 35, in test_add
    case.data = replace(case.data)  # 若存在寫在配置文件中的賬號和密碼、項目名，就必須通過replace函數去拿到正則表達式匹配的值
  File "D:\PycharmProjects\test2-24\cookies0409\UseExcel\common\context.py", line 19, in replace
    da = str(da['id'])                   #從數據庫取出來的id是int
TypeError: 'NoneType' object is not subscriptable


    </pre>
    </div>
    </td>
</tr>

<tr id='ft1_18' class='none'>
    <td class='errorCase'><div class='testcase'>test_add_18</div></td>
    <td colspan='5' align='center'>
    <!--默认收起错误信息 -Findyou
    <button id='btn_ft1_18' type="button"  class="btn btn-danger btn-xs collapsed" data-toggle="collapse" data-target='#div_ft1_18'>错误</button>
    <div id='div_ft1_18' class="collapse">  -->
    <!-- 默认展开错误信息 -Findyou -->
    <button id='btn_ft1_18' type="button"  class="btn btn-danger btn-xs" data-toggle="collapse" data-target='#div_ft1_18'>错误</button>
    <div id='div_ft1_18' class="collapse in" align="left">
    <pre>
    
ft1_18: Traceback (most recent call last):
  File "C:\Python37\lib\site-packages\ddt.py", line 145, in wrapper
    return func(self, *args, **kwargs)
  File "D:\PycharmProjects\test2-24\cookies0409\UseExcel\testcases\test_add.py", line 35, in test_add
    case.data = replace(case.data)  # 若存在寫在配置文件中的賬號和密碼、項目名，就必須通過replace函數去拿到正則表達式匹配的值
  File "D:\PycharmProjects\test2-24\cookies0409\UseExcel\common\context.py", line 19, in replace
    da = str(da['id'])                   #從數據庫取出來的id是int
TypeError: 'NoneType' object is not subscriptable


    </pre>
    </div>
    </td>
</tr>

<tr id='ft1_19' class='none'>
    <td class='errorCase'><div class='testcase'>test_add_19</div></td>
    <td colspan='5' align='center'>
    <!--默认收起错误信息 -Findyou
    <button id='btn_ft1_19' type="button"  class="btn btn-danger btn-xs collapsed" data-toggle="collapse" data-target='#div_ft1_19'>错误</button>
    <div id='div_ft1_19' class="collapse">  -->
    <!-- 默认展开错误信息 -Findyou -->
    <button id='btn_ft1_19' type="button"  class="btn btn-danger btn-xs" data-toggle="collapse" data-target='#div_ft1_19'>错误</button>
    <div id='div_ft1_19' class="collapse in" align="left">
    <pre>
    
ft1_19: Traceback (most recent call last):
  File "C:\Python37\lib\site-packages\ddt.py", line 145, in wrapper
    return func(self, *args, **kwargs)
  File "D:\PycharmProjects\test2-24\cookies0409\UseExcel\testcases\test_add.py", line 35, in test_add
    case.data = replace(case.data)  # 若存在寫在配置文件中的賬號和密碼、項目名，就必須通過replace函數去拿到正則表達式匹配的值
  File "D:\PycharmProjects\test2-24\cookies0409\UseExcel\common\context.py", line 19, in replace
    da = str(da['id'])                   #從數據庫取出來的id是int
TypeError: 'NoneType' object is not subscriptable


    </pre>
    </div>
    </td>
</tr>

<tr id='ft1_20' class='none'>
    <td class='errorCase'><div class='testcase'>test_add_20</div></td>
    <td colspan='5' align='center'>
    <!--默认收起错误信息 -Findyou
    <button id='btn_ft1_20' type="button"  class="btn btn-danger btn-xs collapsed" data-toggle="collapse" data-target='#div_ft1_20'>错误</button>
    <div id='div_ft1_20' class="collapse">  -->
    <!-- 默认展开错误信息 -Findyou -->
    <button id='btn_ft1_20' type="button"  class="btn btn-danger btn-xs" data-toggle="collapse" data-target='#div_ft1_20'>错误</button>
    <div id='div_ft1_20' class="collapse in" align="left">
    <pre>
    
ft1_20: Traceback (most recent call last):
  File "C:\Python37\lib\site-packages\ddt.py", line 145, in wrapper
    return func(self, *args, **kwargs)
  File "D:\PycharmProjects\test2-24\cookies0409\UseExcel\testcases\test_add.py", line 35, in test_add
    case.data = replace(case.data)  # 若存在寫在配置文件中的賬號和密碼、項目名，就必須通過replace函數去拿到正則表達式匹配的值
  File "D:\PycharmProjects\test2-24\cookies0409\UseExcel\common\context.py", line 19, in replace
    da = str(da['id'])                   #從數據庫取出來的id是int
TypeError: 'NoneType' object is not subscriptable


    </pre>
    </div>
    </td>
</tr>

<tr id='ft1_21' class='none'>
    <td class='errorCase'><div class='testcase'>test_add_21</div></td>
    <td colspan='5' align='center'>
    <!--默认收起错误信息 -Findyou
    <button id='btn_ft1_21' type="button"  class="btn btn-danger btn-xs collapsed" data-toggle="collapse" data-target='#div_ft1_21'>错误</button>
    <div id='div_ft1_21' class="collapse">  -->
    <!-- 默认展开错误信息 -Findyou -->
    <button id='btn_ft1_21' type="button"  class="btn btn-danger btn-xs" data-toggle="collapse" data-target='#div_ft1_21'>错误</button>
    <div id='div_ft1_21' class="collapse in" align="left">
    <pre>
    
ft1_21: Traceback (most recent call last):
  File "C:\Python37\lib\site-packages\ddt.py", line 145, in wrapper
    return func(self, *args, **kwargs)
  File "D:\PycharmProjects\test2-24\cookies0409\UseExcel\testcases\test_add.py", line 35, in test_add
    case.data = replace(case.data)  # 若存在寫在配置文件中的賬號和密碼、項目名，就必須通過replace函數去拿到正則表達式匹配的值
  File "D:\PycharmProjects\test2-24\cookies0409\UseExcel\common\context.py", line 19, in replace
    da = str(da['id'])                   #從數據庫取出來的id是int
TypeError: 'NoneType' object is not subscriptable


    </pre>
    </div>
    </td>
</tr>

<tr id='ft1_22' class='none'>
    <td class='errorCase'><div class='testcase'>test_add_22</div></td>
    <td colspan='5' align='center'>
    <!--默认收起错误信息 -Findyou
    <button id='btn_ft1_22' type="button"  class="btn btn-danger btn-xs collapsed" data-toggle="collapse" data-target='#div_ft1_22'>错误</button>
    <div id='div_ft1_22' class="collapse">  -->
    <!-- 默认展开错误信息 -Findyou -->
    <button id='btn_ft1_22' type="button"  class="btn btn-danger btn-xs" data-toggle="collapse" data-target='#div_ft1_22'>错误</button>
    <div id='div_ft1_22' class="collapse in" align="left">
    <pre>
    
ft1_22: Traceback (most recent call last):
  File "C:\Python37\lib\site-packages\ddt.py", line 145, in wrapper
    return func(self, *args, **kwargs)
  File "D:\PycharmProjects\test2-24\cookies0409\UseExcel\testcases\test_add.py", line 35, in test_add
    case.data = replace(case.data)  # 若存在寫在配置文件中的賬號和密碼、項目名，就必須通過replace函數去拿到正則表達式匹配的值
  File "D:\PycharmProjects\test2-24\cookies0409\UseExcel\common\context.py", line 19, in replace
    da = str(da['id'])                   #從數據庫取出來的id是int
TypeError: 'NoneType' object is not subscriptable


    </pre>
    </div>
    </td>
</tr>

<tr id='ft1_23' class='none'>
    <td class='errorCase'><div class='testcase'>test_add_23</div></td>
    <td colspan='5' align='center'>
    <!--默认收起错误信息 -Findyou
    <button id='btn_ft1_23' type="button"  class="btn btn-danger btn-xs collapsed" data-toggle="collapse" data-target='#div_ft1_23'>错误</button>
    <div id='div_ft1_23' class="collapse">  -->
    <!-- 默认展开错误信息 -Findyou -->
    <button id='btn_ft1_23' type="button"  class="btn btn-danger btn-xs" data-toggle="collapse" data-target='#div_ft1_23'>错误</button>
    <div id='div_ft1_23' class="collapse in" align="left">
    <pre>
    
ft1_23: Traceback (most recent call last):
  File "C:\Python37\lib\site-packages\ddt.py", line 145, in wrapper
    return func(self, *args, **kwargs)
  File "D:\PycharmProjects\test2-24\cookies0409\UseExcel\testcases\test_add.py", line 35, in test_add
    case.data = replace(case.data)  # 若存在寫在配置文件中的賬號和密碼、項目名，就必須通過replace函數去拿到正則表達式匹配的值
  File "D:\PycharmProjects\test2-24\cookies0409\UseExcel\common\context.py", line 19, in replace
    da = str(da['id'])                   #從數據庫取出來的id是int
TypeError: 'NoneType' object is not subscriptable


    </pre>
    </div>
    </td>
</tr>

<tr id='ft1_24' class='none'>
    <td class='errorCase'><div class='testcase'>test_add_24</div></td>
    <td colspan='5' align='center'>
    <!--默认收起错误信息 -Findyou
    <button id='btn_ft1_24' type="button"  class="btn btn-danger btn-xs collapsed" data-toggle="collapse" data-target='#div_ft1_24'>错误</button>
    <div id='div_ft1_24' class="collapse">  -->
    <!-- 默认展开错误信息 -Findyou -->
    <button id='btn_ft1_24' type="button"  class="btn btn-danger btn-xs" data-toggle="collapse" data-target='#div_ft1_24'>错误</button>
    <div id='div_ft1_24' class="collapse in" align="left">
    <pre>
    
ft1_24: Traceback (most recent call last):
  File "C:\Python37\lib\site-packages\ddt.py", line 145, in wrapper
    return func(self, *args, **kwargs)
  File "D:\PycharmProjects\test2-24\cookies0409\UseExcel\testcases\test_add.py", line 35, in test_add
    case.data = replace(case.data)  # 若存在寫在配置文件中的賬號和密碼、項目名，就必須通過replace函數去拿到正則表達式匹配的值
  File "D:\PycharmProjects\test2-24\cookies0409\UseExcel\common\context.py", line 19, in replace
    da = str(da['id'])                   #從數據庫取出來的id是int
TypeError: 'NoneType' object is not subscriptable


    </pre>
    </div>
    </td>
</tr>

<tr id='ft1_25' class='none'>
    <td class='errorCase'><div class='testcase'>test_add_25</div></td>
    <td colspan='5' align='center'>
    <!--默认收起错误信息 -Findyou
    <button id='btn_ft1_25' type="button"  class="btn btn-danger btn-xs collapsed" data-toggle="collapse" data-target='#div_ft1_25'>错误</button>
    <div id='div_ft1_25' class="collapse">  -->
    <!-- 默认展开错误信息 -Findyou -->
    <button id='btn_ft1_25' type="button"  class="btn btn-danger btn-xs" data-toggle="collapse" data-target='#div_ft1_25'>错误</button>
    <div id='div_ft1_25' class="collapse in" align="left">
    <pre>
    
ft1_25: Traceback (most recent call last):
  File "C:\Python37\lib\site-packages\ddt.py", line 145, in wrapper
    return func(self, *args, **kwargs)
  File "D:\PycharmProjects\test2-24\cookies0409\UseExcel\testcases\test_add.py", line 35, in test_add
    case.data = replace(case.data)  # 若存在寫在配置文件中的賬號和密碼、項目名，就必須通過replace函數去拿到正則表達式匹配的值
  File "D:\PycharmProjects\test2-24\cookies0409\UseExcel\common\context.py", line 19, in replace
    da = str(da['id'])                   #從數據庫取出來的id是int
TypeError: 'NoneType' object is not subscriptable


    </pre>
    </div>
    </td>
</tr>

<tr id='ft1_26' class='none'>
    <td class='errorCase'><div class='testcase'>test_add_26</div></td>
    <td colspan='5' align='center'>
    <!--默认收起错误信息 -Findyou
    <button id='btn_ft1_26' type="button"  class="btn btn-danger btn-xs collapsed" data-toggle="collapse" data-target='#div_ft1_26'>错误</button>
    <div id='div_ft1_26' class="collapse">  -->
    <!-- 默认展开错误信息 -Findyou -->
    <button id='btn_ft1_26' type="button"  class="btn btn-danger btn-xs" data-toggle="collapse" data-target='#div_ft1_26'>错误</button>
    <div id='div_ft1_26' class="collapse in" align="left">
    <pre>
    
ft1_26: Traceback (most recent call last):
  File "C:\Python37\lib\site-packages\ddt.py", line 145, in wrapper
    return func(self, *args, **kwargs)
  File "D:\PycharmProjects\test2-24\cookies0409\UseExcel\testcases\test_add.py", line 35, in test_add
    case.data = replace(case.data)  # 若存在寫在配置文件中的賬號和密碼、項目名，就必須通過replace函數去拿到正則表達式匹配的值
  File "D:\PycharmProjects\test2-24\cookies0409\UseExcel\common\context.py", line 19, in replace
    da = str(da['id'])                   #從數據庫取出來的id是int
TypeError: 'NoneType' object is not subscriptable


    </pre>
    </div>
    </td>
</tr>

<tr id='ft1_27' class='none'>
    <td class='errorCase'><div class='testcase'>test_add_27</div></td>
    <td colspan='5' align='center'>
    <!--默认收起错误信息 -Findyou
    <button id='btn_ft1_27' type="button"  class="btn btn-danger btn-xs collapsed" data-toggle="collapse" data-target='#div_ft1_27'>错误</button>
    <div id='div_ft1_27' class="collapse">  -->
    <!-- 默认展开错误信息 -Findyou -->
    <button id='btn_ft1_27' type="button"  class="btn btn-danger btn-xs" data-toggle="collapse" data-target='#div_ft1_27'>错误</button>
    <div id='div_ft1_27' class="collapse in" align="left">
    <pre>
    
ft1_27: Traceback (most recent call last):
  File "C:\Python37\lib\site-packages\ddt.py", line 145, in wrapper
    return func(self, *args, **kwargs)
  File "D:\PycharmProjects\test2-24\cookies0409\UseExcel\testcases\test_add.py", line 35, in test_add
    case.data = replace(case.data)  # 若存在寫在配置文件中的賬號和密碼、項目名，就必須通過replace函數去拿到正則表達式匹配的值
  File "D:\PycharmProjects\test2-24\cookies0409\UseExcel\common\context.py", line 19, in replace
    da = str(da['id'])                   #從數據庫取出來的id是int
TypeError: 'NoneType' object is not subscriptable


    </pre>
    </div>
    </td>
</tr>

<tr id='ft1_28' class='none'>
    <td class='errorCase'><div class='testcase'>test_add_28</div></td>
    <td colspan='5' align='center'>
    <!--默认收起错误信息 -Findyou
    <button id='btn_ft1_28' type="button"  class="btn btn-danger btn-xs collapsed" data-toggle="collapse" data-target='#div_ft1_28'>错误</button>
    <div id='div_ft1_28' class="collapse">  -->
    <!-- 默认展开错误信息 -Findyou -->
    <button id='btn_ft1_28' type="button"  class="btn btn-danger btn-xs" data-toggle="collapse" data-target='#div_ft1_28'>错误</button>
    <div id='div_ft1_28' class="collapse in" align="left">
    <pre>
    
ft1_28: Traceback (most recent call last):
  File "C:\Python37\lib\site-packages\ddt.py", line 145, in wrapper
    return func(self, *args, **kwargs)
  File "D:\PycharmProjects\test2-24\cookies0409\UseExcel\testcases\test_add.py", line 35, in test_add
    case.data = replace(case.data)  # 若存在寫在配置文件中的賬號和密碼、項目名，就必須通過replace函數去拿到正則表達式匹配的值
  File "D:\PycharmProjects\test2-24\cookies0409\UseExcel\common\context.py", line 19, in replace
    da = str(da['id'])                   #從數據庫取出來的id是int
TypeError: 'NoneType' object is not subscriptable


    </pre>
    </div>
    </td>
</tr>

<tr id='ft1_29' class='none'>
    <td class='errorCase'><div class='testcase'>test_add_29</div></td>
    <td colspan='5' align='center'>
    <!--默认收起错误信息 -Findyou
    <button id='btn_ft1_29' type="button"  class="btn btn-danger btn-xs collapsed" data-toggle="collapse" data-target='#div_ft1_29'>错误</button>
    <div id='div_ft1_29' class="collapse">  -->
    <!-- 默认展开错误信息 -Findyou -->
    <button id='btn_ft1_29' type="button"  class="btn btn-danger btn-xs" data-toggle="collapse" data-target='#div_ft1_29'>错误</button>
    <div id='div_ft1_29' class="collapse in" align="left">
    <pre>
    
ft1_29: Traceback (most recent call last):
  File "C:\Python37\lib\site-packages\ddt.py", line 145, in wrapper
    return func(self, *args, **kwargs)
  File "D:\PycharmProjects\test2-24\cookies0409\UseExcel\testcases\test_add.py", line 35, in test_add
    case.data = replace(case.data)  # 若存在寫在配置文件中的賬號和密碼、項目名，就必須通過replace函數去拿到正則表達式匹配的值
  File "D:\PycharmProjects\test2-24\cookies0409\UseExcel\common\context.py", line 19, in replace
    da = str(da['id'])                   #從數據庫取出來的id是int
TypeError: 'NoneType' object is not subscriptable


    </pre>
    </div>
    </td>
</tr>

<tr id='ft1_30' class='none'>
    <td class='errorCase'><div class='testcase'>test_add_30</div></td>
    <td colspan='5' align='center'>
    <!--默认收起错误信息 -Findyou
    <button id='btn_ft1_30' type="button"  class="btn btn-danger btn-xs collapsed" data-toggle="collapse" data-target='#div_ft1_30'>错误</button>
    <div id='div_ft1_30' class="collapse">  -->
    <!-- 默认展开错误信息 -Findyou -->
    <button id='btn_ft1_30' type="button"  class="btn btn-danger btn-xs" data-toggle="collapse" data-target='#div_ft1_30'>错误</button>
    <div id='div_ft1_30' class="collapse in" align="left">
    <pre>
    
ft1_30: Traceback (most recent call last):
  File "C:\Python37\lib\site-packages\ddt.py", line 145, in wrapper
    return func(self, *args, **kwargs)
  File "D:\PycharmProjects\test2-24\cookies0409\UseExcel\testcases\test_add.py", line 35, in test_add
    case.data = replace(case.data)  # 若存在寫在配置文件中的賬號和密碼、項目名，就必須通過replace函數去拿到正則表達式匹配的值
  File "D:\PycharmProjects\test2-24\cookies0409\UseExcel\common\context.py", line 19, in replace
    da = str(da['id'])                   #從數據庫取出來的id是int
TypeError: 'NoneType' object is not subscriptable


    </pre>
    </div>
    </td>
</tr>

<tr id='ft1_31' class='none'>
    <td class='errorCase'><div class='testcase'>test_add_31</div></td>
    <td colspan='5' align='center'>
    <!--默认收起错误信息 -Findyou
    <button id='btn_ft1_31' type="button"  class="btn btn-danger btn-xs collapsed" data-toggle="collapse" data-target='#div_ft1_31'>错误</button>
    <div id='div_ft1_31' class="collapse">  -->
    <!-- 默认展开错误信息 -Findyou -->
    <button id='btn_ft1_31' type="button"  class="btn btn-danger btn-xs" data-toggle="collapse" data-target='#div_ft1_31'>错误</button>
    <div id='div_ft1_31' class="collapse in" align="left">
    <pre>
    
ft1_31: Traceback (most recent call last):
  File "C:\Python37\lib\site-packages\ddt.py", line 145, in wrapper
    return func(self, *args, **kwargs)
  File "D:\PycharmProjects\test2-24\cookies0409\UseExcel\testcases\test_add.py", line 35, in test_add
    case.data = replace(case.data)  # 若存在寫在配置文件中的賬號和密碼、項目名，就必須通過replace函數去拿到正則表達式匹配的值
  File "D:\PycharmProjects\test2-24\cookies0409\UseExcel\common\context.py", line 19, in replace
    da = str(da['id'])                   #從數據庫取出來的id是int
TypeError: 'NoneType' object is not subscriptable


    </pre>
    </div>
    </td>
</tr>

<tr id='ft1_32' class='none'>
    <td class='errorCase'><div class='testcase'>test_add_32</div></td>
    <td colspan='5' align='center'>
    <!--默认收起错误信息 -Findyou
    <button id='btn_ft1_32' type="button"  class="btn btn-danger btn-xs collapsed" data-toggle="collapse" data-target='#div_ft1_32'>错误</button>
    <div id='div_ft1_32' class="collapse">  -->
    <!-- 默认展开错误信息 -Findyou -->
    <button id='btn_ft1_32' type="button"  class="btn btn-danger btn-xs" data-toggle="collapse" data-target='#div_ft1_32'>错误</button>
    <div id='div_ft1_32' class="collapse in" align="left">
    <pre>
    
ft1_32: Traceback (most recent call last):
  File "C:\Python37\lib\site-packages\ddt.py", line 145, in wrapper
    return func(self, *args, **kwargs)
  File "D:\PycharmProjects\test2-24\cookies0409\UseExcel\testcases\test_add.py", line 35, in test_add
    case.data = replace(case.data)  # 若存在寫在配置文件中的賬號和密碼、項目名，就必須通過replace函數去拿到正則表達式匹配的值
  File "D:\PycharmProjects\test2-24\cookies0409\UseExcel\common\context.py", line 19, in replace
    da = str(da['id'])                   #從數據庫取出來的id是int
TypeError: 'NoneType' object is not subscriptable


    </pre>
    </div>
    </td>
</tr>

<tr id='ft1_33' class='none'>
    <td class='errorCase'><div class='testcase'>test_add_33</div></td>
    <td colspan='5' align='center'>
    <!--默认收起错误信息 -Findyou
    <button id='btn_ft1_33' type="button"  class="btn btn-danger btn-xs collapsed" data-toggle="collapse" data-target='#div_ft1_33'>错误</button>
    <div id='div_ft1_33' class="collapse">  -->
    <!-- 默认展开错误信息 -Findyou -->
    <button id='btn_ft1_33' type="button"  class="btn btn-danger btn-xs" data-toggle="collapse" data-target='#div_ft1_33'>错误</button>
    <div id='div_ft1_33' class="collapse in" align="left">
    <pre>
    
ft1_33: Traceback (most recent call last):
  File "C:\Python37\lib\site-packages\ddt.py", line 145, in wrapper
    return func(self, *args, **kwargs)
  File "D:\PycharmProjects\test2-24\cookies0409\UseExcel\testcases\test_add.py", line 35, in test_add
    case.data = replace(case.data)  # 若存在寫在配置文件中的賬號和密碼、項目名，就必須通過replace函數去拿到正則表達式匹配的值
  File "D:\PycharmProjects\test2-24\cookies0409\UseExcel\common\context.py", line 19, in replace
    da = str(da['id'])                   #從數據庫取出來的id是int
TypeError: 'NoneType' object is not subscriptable


    </pre>
    </div>
    </td>
</tr>

<tr id='ft1_34' class='none'>
    <td class='errorCase'><div class='testcase'>test_add_34</div></td>
    <td colspan='5' align='center'>
    <!--默认收起错误信息 -Findyou
    <button id='btn_ft1_34' type="button"  class="btn btn-danger btn-xs collapsed" data-toggle="collapse" data-target='#div_ft1_34'>错误</button>
    <div id='div_ft1_34' class="collapse">  -->
    <!-- 默认展开错误信息 -Findyou -->
    <button id='btn_ft1_34' type="button"  class="btn btn-danger btn-xs" data-toggle="collapse" data-target='#div_ft1_34'>错误</button>
    <div id='div_ft1_34' class="collapse in" align="left">
    <pre>
    
ft1_34: Traceback (most recent call last):
  File "C:\Python37\lib\site-packages\ddt.py", line 145, in wrapper
    return func(self, *args, **kwargs)
  File "D:\PycharmProjects\test2-24\cookies0409\UseExcel\testcases\test_add.py", line 35, in test_add
    case.data = replace(case.data)  # 若存在寫在配置文件中的賬號和密碼、項目名，就必須通過replace函數去拿到正則表達式匹配的值
  File "D:\PycharmProjects\test2-24\cookies0409\UseExcel\common\context.py", line 19, in replace
    da = str(da['id'])                   #從數據庫取出來的id是int
TypeError: 'NoneType' object is not subscriptable


    </pre>
    </div>
    </td>
</tr>

<tr id='ft1_35' class='none'>
    <td class='errorCase'><div class='testcase'>test_add_35</div></td>
    <td colspan='5' align='center'>
    <!--默认收起错误信息 -Findyou
    <button id='btn_ft1_35' type="button"  class="btn btn-danger btn-xs collapsed" data-toggle="collapse" data-target='#div_ft1_35'>错误</button>
    <div id='div_ft1_35' class="collapse">  -->
    <!-- 默认展开错误信息 -Findyou -->
    <button id='btn_ft1_35' type="button"  class="btn btn-danger btn-xs" data-toggle="collapse" data-target='#div_ft1_35'>错误</button>
    <div id='div_ft1_35' class="collapse in" align="left">
    <pre>
    
ft1_35: Traceback (most recent call last):
  File "C:\Python37\lib\site-packages\ddt.py", line 145, in wrapper
    return func(self, *args, **kwargs)
  File "D:\PycharmProjects\test2-24\cookies0409\UseExcel\testcases\test_add.py", line 35, in test_add
    case.data = replace(case.data)  # 若存在寫在配置文件中的賬號和密碼、項目名，就必須通過replace函數去拿到正則表達式匹配的值
  File "D:\PycharmProjects\test2-24\cookies0409\UseExcel\common\context.py", line 19, in replace
    da = str(da['id'])                   #從數據庫取出來的id是int
TypeError: 'NoneType' object is not subscriptable


    </pre>
    </div>
    </td>
</tr>

<tr id='ft1_36' class='none'>
    <td class='errorCase'><div class='testcase'>test_add_36</div></td>
    <td colspan='5' align='center'>
    <!--默认收起错误信息 -Findyou
    <button id='btn_ft1_36' type="button"  class="btn btn-danger btn-xs collapsed" data-toggle="collapse" data-target='#div_ft1_36'>错误</button>
    <div id='div_ft1_36' class="collapse">  -->
    <!-- 默认展开错误信息 -Findyou -->
    <button id='btn_ft1_36' type="button"  class="btn btn-danger btn-xs" data-toggle="collapse" data-target='#div_ft1_36'>错误</button>
    <div id='div_ft1_36' class="collapse in" align="left">
    <pre>
    
ft1_36: Traceback (most recent call last):
  File "C:\Python37\lib\site-packages\ddt.py", line 145, in wrapper
    return func(self, *args, **kwargs)
  File "D:\PycharmProjects\test2-24\cookies0409\UseExcel\testcases\test_add.py", line 35, in test_add
    case.data = replace(case.data)  # 若存在寫在配置文件中的賬號和密碼、項目名，就必須通過replace函數去拿到正則表達式匹配的值
  File "D:\PycharmProjects\test2-24\cookies0409\UseExcel\common\context.py", line 19, in replace
    da = str(da['id'])                   #從數據庫取出來的id是int
TypeError: 'NoneType' object is not subscriptable


    </pre>
    </div>
    </td>
</tr>

<tr id='ft1_37' class='none'>
    <td class='errorCase'><div class='testcase'>test_add_37</div></td>
    <td colspan='5' align='center'>
    <!--默认收起错误信息 -Findyou
    <button id='btn_ft1_37' type="button"  class="btn btn-danger btn-xs collapsed" data-toggle="collapse" data-target='#div_ft1_37'>错误</button>
    <div id='div_ft1_37' class="collapse">  -->
    <!-- 默认展开错误信息 -Findyou -->
    <button id='btn_ft1_37' type="button"  class="btn btn-danger btn-xs" data-toggle="collapse" data-target='#div_ft1_37'>错误</button>
    <div id='div_ft1_37' class="collapse in" align="left">
    <pre>
    
ft1_37: Traceback (most recent call last):
  File "C:\Python37\lib\site-packages\ddt.py", line 145, in wrapper
    return func(self, *args, **kwargs)
  File "D:\PycharmProjects\test2-24\cookies0409\UseExcel\testcases\test_add.py", line 35, in test_add
    case.data = replace(case.data)  # 若存在寫在配置文件中的賬號和密碼、項目名，就必須通過replace函數去拿到正則表達式匹配的值
  File "D:\PycharmProjects\test2-24\cookies0409\UseExcel\common\context.py", line 19, in replace
    da = str(da['id'])                   #從數據庫取出來的id是int
TypeError: 'NoneType' object is not subscriptable


    </pre>
    </div>
    </td>
</tr>

<tr id='ft1_38' class='none'>
    <td class='errorCase'><div class='testcase'>test_add_38</div></td>
    <td colspan='5' align='center'>
    <!--默认收起错误信息 -Findyou
    <button id='btn_ft1_38' type="button"  class="btn btn-danger btn-xs collapsed" data-toggle="collapse" data-target='#div_ft1_38'>错误</button>
    <div id='div_ft1_38' class="collapse">  -->
    <!-- 默认展开错误信息 -Findyou -->
    <button id='btn_ft1_38' type="button"  class="btn btn-danger btn-xs" data-toggle="collapse" data-target='#div_ft1_38'>错误</button>
    <div id='div_ft1_38' class="collapse in" align="left">
    <pre>
    
ft1_38: Traceback (most recent call last):
  File "C:\Python37\lib\site-packages\ddt.py", line 145, in wrapper
    return func(self, *args, **kwargs)
  File "D:\PycharmProjects\test2-24\cookies0409\UseExcel\testcases\test_add.py", line 35, in test_add
    case.data = replace(case.data)  # 若存在寫在配置文件中的賬號和密碼、項目名，就必須通過replace函數去拿到正則表達式匹配的值
  File "D:\PycharmProjects\test2-24\cookies0409\UseExcel\common\context.py", line 19, in replace
    da = str(da['id'])                   #從數據庫取出來的id是int
TypeError: 'NoneType' object is not subscriptable


    </pre>
    </div>
    </td>
</tr>

<tr id='ft1_39' class='none'>
    <td class='errorCase'><div class='testcase'>test_add_39</div></td>
    <td colspan='5' align='center'>
    <!--默认收起错误信息 -Findyou
    <button id='btn_ft1_39' type="button"  class="btn btn-danger btn-xs collapsed" data-toggle="collapse" data-target='#div_ft1_39'>错误</button>
    <div id='div_ft1_39' class="collapse">  -->
    <!-- 默认展开错误信息 -Findyou -->
    <button id='btn_ft1_39' type="button"  class="btn btn-danger btn-xs" data-toggle="collapse" data-target='#div_ft1_39'>错误</button>
    <div id='div_ft1_39' class="collapse in" align="left">
    <pre>
    
ft1_39: Traceback (most recent call last):
  File "C:\Python37\lib\site-packages\ddt.py", line 145, in wrapper
    return func(self, *args, **kwargs)
  File "D:\PycharmProjects\test2-24\cookies0409\UseExcel\testcases\test_add.py", line 35, in test_add
    case.data = replace(case.data)  # 若存在寫在配置文件中的賬號和密碼、項目名，就必須通過replace函數去拿到正則表達式匹配的值
  File "D:\PycharmProjects\test2-24\cookies0409\UseExcel\common\context.py", line 19, in replace
    da = str(da['id'])                   #從數據庫取出來的id是int
TypeError: 'NoneType' object is not subscriptable


    </pre>
    </div>
    </td>
</tr>

<tr id='ft1_40' class='none'>
    <td class='errorCase'><div class='testcase'>test_add_40</div></td>
    <td colspan='5' align='center'>
    <!--默认收起错误信息 -Findyou
    <button id='btn_ft1_40' type="button"  class="btn btn-danger btn-xs collapsed" data-toggle="collapse" data-target='#div_ft1_40'>错误</button>
    <div id='div_ft1_40' class="collapse">  -->
    <!-- 默认展开错误信息 -Findyou -->
    <button id='btn_ft1_40' type="button"  class="btn btn-danger btn-xs" data-toggle="collapse" data-target='#div_ft1_40'>错误</button>
    <div id='div_ft1_40' class="collapse in" align="left">
    <pre>
    
ft1_40: Traceback (most recent call last):
  File "C:\Python37\lib\site-packages\ddt.py", line 145, in wrapper
    return func(self, *args, **kwargs)
  File "D:\PycharmProjects\test2-24\cookies0409\UseExcel\testcases\test_add.py", line 35, in test_add
    case.data = replace(case.data)  # 若存在寫在配置文件中的賬號和密碼、項目名，就必須通過replace函數去拿到正則表達式匹配的值
  File "D:\PycharmProjects\test2-24\cookies0409\UseExcel\common\context.py", line 19, in replace
    da = str(da['id'])                   #從數據庫取出來的id是int
TypeError: 'NoneType' object is not subscriptable


    </pre>
    </div>
    </td>
</tr>

<tr id='ft1_41' class='none'>
    <td class='errorCase'><div class='testcase'>test_add_41</div></td>
    <td colspan='5' align='center'>
    <!--默认收起错误信息 -Findyou
    <button id='btn_ft1_41' type="button"  class="btn btn-danger btn-xs collapsed" data-toggle="collapse" data-target='#div_ft1_41'>错误</button>
    <div id='div_ft1_41' class="collapse">  -->
    <!-- 默认展开错误信息 -Findyou -->
    <button id='btn_ft1_41' type="button"  class="btn btn-danger btn-xs" data-toggle="collapse" data-target='#div_ft1_41'>错误</button>
    <div id='div_ft1_41' class="collapse in" align="left">
    <pre>
    
ft1_41: Traceback (most recent call last):
  File "C:\Python37\lib\site-packages\ddt.py", line 145, in wrapper
    return func(self, *args, **kwargs)
  File "D:\PycharmProjects\test2-24\cookies0409\UseExcel\testcases\test_add.py", line 35, in test_add
    case.data = replace(case.data)  # 若存在寫在配置文件中的賬號和密碼、項目名，就必須通過replace函數去拿到正則表達式匹配的值
  File "D:\PycharmProjects\test2-24\cookies0409\UseExcel\common\context.py", line 19, in replace
    da = str(da['id'])                   #從數據庫取出來的id是int
TypeError: 'NoneType' object is not subscriptable


    </pre>
    </div>
    </td>
</tr>

<tr id='ft1_42' class='none'>
    <td class='errorCase'><div class='testcase'>test_add_42</div></td>
    <td colspan='5' align='center'>
    <!--默认收起错误信息 -Findyou
    <button id='btn_ft1_42' type="button"  class="btn btn-danger btn-xs collapsed" data-toggle="collapse" data-target='#div_ft1_42'>错误</button>
    <div id='div_ft1_42' class="collapse">  -->
    <!-- 默认展开错误信息 -Findyou -->
    <button id='btn_ft1_42' type="button"  class="btn btn-danger btn-xs" data-toggle="collapse" data-target='#div_ft1_42'>错误</button>
    <div id='div_ft1_42' class="collapse in" align="left">
    <pre>
    
ft1_42: Traceback (most recent call last):
  File "C:\Python37\lib\site-packages\ddt.py", line 145, in wrapper
    return func(self, *args, **kwargs)
  File "D:\PycharmProjects\test2-24\cookies0409\UseExcel\testcases\test_add.py", line 35, in test_add
    case.data = replace(case.data)  # 若存在寫在配置文件中的賬號和密碼、項目名，就必須通過replace函數去拿到正則表達式匹配的值
  File "D:\PycharmProjects\test2-24\cookies0409\UseExcel\common\context.py", line 19, in replace
    da = str(da['id'])                   #從數據庫取出來的id是int
TypeError: 'NoneType' object is not subscriptable


    </pre>
    </div>
    </td>
</tr>

<tr id='ft1_43' class='none'>
    <td class='errorCase'><div class='testcase'>test_add_43</div></td>
    <td colspan='5' align='center'>
    <!--默认收起错误信息 -Findyou
    <button id='btn_ft1_43' type="button"  class="btn btn-danger btn-xs collapsed" data-toggle="collapse" data-target='#div_ft1_43'>错误</button>
    <div id='div_ft1_43' class="collapse">  -->
    <!-- 默认展开错误信息 -Findyou -->
    <button id='btn_ft1_43' type="button"  class="btn btn-danger btn-xs" data-toggle="collapse" data-target='#div_ft1_43'>错误</button>
    <div id='div_ft1_43' class="collapse in" align="left">
    <pre>
    
ft1_43: Traceback (most recent call last):
  File "C:\Python37\lib\site-packages\ddt.py", line 145, in wrapper
    return func(self, *args, **kwargs)
  File "D:\PycharmProjects\test2-24\cookies0409\UseExcel\testcases\test_add.py", line 35, in test_add
    case.data = replace(case.data)  # 若存在寫在配置文件中的賬號和密碼、項目名，就必須通過replace函數去拿到正則表達式匹配的值
  File "D:\PycharmProjects\test2-24\cookies0409\UseExcel\common\context.py", line 19, in replace
    da = str(da['id'])                   #從數據庫取出來的id是int
TypeError: 'NoneType' object is not subscriptable


    </pre>
    </div>
    </td>
</tr>

<tr class='failClass warning'>
    <td>test_audit.TestAudit: 測試審批流程</td>
    <td class="text-center">36</td>
    <td class="text-center">10</td>
    <td class="text-center">26</td>
    <td class="text-center">0</td>
    <td class="text-center"><a href="javascript:showClassDetail('c2',36)" class="detail" id='c2'>详细</a></td>
</tr>

<tr id='pt2_1' class='hiddenRow'>
    <td class='passCase'><div class='testcase'>test_audit_01</div></td>
    <td colspan='5' align='center'><span class="label label-success success">通过</span></td>
</tr>

<tr id='pt2_2' class='hiddenRow'>
    <td class='passCase'><div class='testcase'>test_audit_02</div></td>
    <td colspan='5' align='center'><span class="label label-success success">通过</span></td>
</tr>

<tr id='pt2_3' class='hiddenRow'>
    <td class='passCase'><div class='testcase'>test_audit_03</div></td>
    <td colspan='5' align='center'><span class="label label-success success">通过</span></td>
</tr>

<tr id='pt2_4' class='hiddenRow'>
    <td class='passCase'><div class='testcase'>test_audit_04</div></td>
    <td colspan='5' align='center'><span class="label label-success success">通过</span></td>
</tr>

<tr id='pt2_5' class='hiddenRow'>
    <td class='passCase'><div class='testcase'>test_audit_05</div></td>
    <td colspan='5' align='center'><span class="label label-success success">通过</span></td>
</tr>

<tr id='pt2_6' class='hiddenRow'>
    <td class='passCase'><div class='testcase'>test_audit_06</div></td>
    <td colspan='5' align='center'><span class="label label-success success">通过</span></td>
</tr>

<tr id='pt2_7' class='hiddenRow'>
    <td class='passCase'><div class='testcase'>test_audit_07</div></td>
    <td colspan='5' align='center'><span class="label label-success success">通过</span></td>
</tr>

<tr id='pt2_8' class='hiddenRow'>
    <td class='passCase'><div class='testcase'>test_audit_08</div></td>
    <td colspan='5' align='center'><span class="label label-success success">通过</span></td>
</tr>

<tr id='pt2_9' class='hiddenRow'>
    <td class='passCase'><div class='testcase'>test_audit_09</div></td>
    <td colspan='5' align='center'><span class="label label-success success">通过</span></td>
</tr>

<tr id='pt2_10' class='hiddenRow'>
    <td class='passCase'><div class='testcase'>test_audit_10</div></td>
    <td colspan='5' align='center'><span class="label label-success success">通过</span></td>
</tr>

<tr id='ft2_11' class='none'>
    <td class='failCase'><div class='testcase'>test_audit_11</div></td>
    <td colspan='5' align='center'>
    <!--默认收起错误信息 -Findyou
    <button id='btn_ft2_11' type="button"  class="btn btn-danger btn-xs collapsed" data-toggle="collapse" data-target='#div_ft2_11'>失败</button>
    <div id='div_ft2_11' class="collapse">  -->
    <!-- 默认展开错误信息 -Findyou -->
    <button id='btn_ft2_11' type="button"  class="btn btn-danger btn-xs" data-toggle="collapse" data-target='#div_ft2_11'>失败</button>
    <div id='div_ft2_11' class="collapse in" align="left">
    <pre>
    
ft2_11: Traceback (most recent call last):
  File "C:\Python37\lib\site-packages\ddt.py", line 145, in wrapper
    return func(self, *args, **kwargs)
  File "D:\PycharmProjects\test2-24\cookies0409\UseExcel\testcases\test_audit.py", line 40, in test_audit
    raise e                                            #沒有寫這個只會顯示用例執行次數，寫之後控制台顯示有誤信息
  File "D:\PycharmProjects\test2-24\cookies0409\UseExcel\testcases\test_audit.py", line 35, in test_audit
    self.unit.assertEqual(str(case.expectid),code )
AssertionError: '20207' != '20206'
- 20207
?     ^
+ 20206
?     ^



    </pre>
    </div>
    </td>
</tr>

<tr id='ft2_12' class='none'>
    <td class='failCase'><div class='testcase'>test_audit_12</div></td>
    <td colspan='5' align='center'>
    <!--默认收起错误信息 -Findyou
    <button id='btn_ft2_12' type="button"  class="btn btn-danger btn-xs collapsed" data-toggle="collapse" data-target='#div_ft2_12'>失败</button>
    <div id='div_ft2_12' class="collapse">  -->
    <!-- 默认展开错误信息 -Findyou -->
    <button id='btn_ft2_12' type="button"  class="btn btn-danger btn-xs" data-toggle="collapse" data-target='#div_ft2_12'>失败</button>
    <div id='div_ft2_12' class="collapse in" align="left">
    <pre>
    
ft2_12: Traceback (most recent call last):
  File "C:\Python37\lib\site-packages\ddt.py", line 145, in wrapper
    return func(self, *args, **kwargs)
  File "D:\PycharmProjects\test2-24\cookies0409\UseExcel\testcases\test_audit.py", line 40, in test_audit
    raise e                                            #沒有寫這個只會顯示用例執行次數，寫之後控制台顯示有誤信息
  File "D:\PycharmProjects\test2-24\cookies0409\UseExcel\testcases\test_audit.py", line 35, in test_audit
    self.unit.assertEqual(str(case.expectid),code )
AssertionError: '20207' != '20206'
- 20207
?     ^
+ 20206
?     ^



    </pre>
    </div>
    </td>
</tr>

<tr id='ft2_13' class='none'>
    <td class='failCase'><div class='testcase'>test_audit_13</div></td>
    <td colspan='5' align='center'>
    <!--默认收起错误信息 -Findyou
    <button id='btn_ft2_13' type="button"  class="btn btn-danger btn-xs collapsed" data-toggle="collapse" data-target='#div_ft2_13'>失败</button>
    <div id='div_ft2_13' class="collapse">  -->
    <!-- 默认展开错误信息 -Findyou -->
    <button id='btn_ft2_13' type="button"  class="btn btn-danger btn-xs" data-toggle="collapse" data-target='#div_ft2_13'>失败</button>
    <div id='div_ft2_13' class="collapse in" align="left">
    <pre>
    
ft2_13: Traceback (most recent call last):
  File "C:\Python37\lib\site-packages\ddt.py", line 145, in wrapper
    return func(self, *args, **kwargs)
  File "D:\PycharmProjects\test2-24\cookies0409\UseExcel\testcases\test_audit.py", line 40, in test_audit
    raise e                                            #沒有寫這個只會顯示用例執行次數，寫之後控制台顯示有誤信息
  File "D:\PycharmProjects\test2-24\cookies0409\UseExcel\testcases\test_audit.py", line 35, in test_audit
    self.unit.assertEqual(str(case.expectid),code )
AssertionError: '10001' != '20206'
- 10001
+ 20206



    </pre>
    </div>
    </td>
</tr>

<tr id='ft2_14' class='none'>
    <td class='failCase'><div class='testcase'>test_audit_14</div></td>
    <td colspan='5' align='center'>
    <!--默认收起错误信息 -Findyou
    <button id='btn_ft2_14' type="button"  class="btn btn-danger btn-xs collapsed" data-toggle="collapse" data-target='#div_ft2_14'>失败</button>
    <div id='div_ft2_14' class="collapse">  -->
    <!-- 默认展开错误信息 -Findyou -->
    <button id='btn_ft2_14' type="button"  class="btn btn-danger btn-xs" data-toggle="collapse" data-target='#div_ft2_14'>失败</button>
    <div id='div_ft2_14' class="collapse in" align="left">
    <pre>
    
ft2_14: Traceback (most recent call last):
  File "C:\Python37\lib\site-packages\ddt.py", line 145, in wrapper
    return func(self, *args, **kwargs)
  File "D:\PycharmProjects\test2-24\cookies0409\UseExcel\testcases\test_audit.py", line 40, in test_audit
    raise e                                            #沒有寫這個只會顯示用例執行次數，寫之後控制台顯示有誤信息
  File "D:\PycharmProjects\test2-24\cookies0409\UseExcel\testcases\test_audit.py", line 35, in test_audit
    self.unit.assertEqual(str(case.expectid),code )
AssertionError: '10001' != '20206'
- 10001
+ 20206



    </pre>
    </div>
    </td>
</tr>

<tr id='ft2_15' class='none'>
    <td class='failCase'><div class='testcase'>test_audit_15</div></td>
    <td colspan='5' align='center'>
    <!--默认收起错误信息 -Findyou
    <button id='btn_ft2_15' type="button"  class="btn btn-danger btn-xs collapsed" data-toggle="collapse" data-target='#div_ft2_15'>失败</button>
    <div id='div_ft2_15' class="collapse">  -->
    <!-- 默认展开错误信息 -Findyou -->
    <button id='btn_ft2_15' type="button"  class="btn btn-danger btn-xs" data-toggle="collapse" data-target='#div_ft2_15'>失败</button>
    <div id='div_ft2_15' class="collapse in" align="left">
    <pre>
    
ft2_15: Traceback (most recent call last):
  File "C:\Python37\lib\site-packages\ddt.py", line 145, in wrapper
    return func(self, *args, **kwargs)
  File "D:\PycharmProjects\test2-24\cookies0409\UseExcel\testcases\test_audit.py", line 40, in test_audit
    raise e                                            #沒有寫這個只會顯示用例執行次數，寫之後控制台顯示有誤信息
  File "D:\PycharmProjects\test2-24\cookies0409\UseExcel\testcases\test_audit.py", line 35, in test_audit
    self.unit.assertEqual(str(case.expectid),code )
AssertionError: '10001' != '20206'
- 10001
+ 20206



    </pre>
    </div>
    </td>
</tr>

<tr id='ft2_16' class='none'>
    <td class='failCase'><div class='testcase'>test_audit_16</div></td>
    <td colspan='5' align='center'>
    <!--默认收起错误信息 -Findyou
    <button id='btn_ft2_16' type="button"  class="btn btn-danger btn-xs collapsed" data-toggle="collapse" data-target='#div_ft2_16'>失败</button>
    <div id='div_ft2_16' class="collapse">  -->
    <!-- 默认展开错误信息 -Findyou -->
    <button id='btn_ft2_16' type="button"  class="btn btn-danger btn-xs" data-toggle="collapse" data-target='#div_ft2_16'>失败</button>
    <div id='div_ft2_16' class="collapse in" align="left">
    <pre>
    
ft2_16: Traceback (most recent call last):
  File "C:\Python37\lib\site-packages\ddt.py", line 145, in wrapper
    return func(self, *args, **kwargs)
  File "D:\PycharmProjects\test2-24\cookies0409\UseExcel\testcases\test_audit.py", line 40, in test_audit
    raise e                                            #沒有寫這個只會顯示用例執行次數，寫之後控制台顯示有誤信息
  File "D:\PycharmProjects\test2-24\cookies0409\UseExcel\testcases\test_audit.py", line 35, in test_audit
    self.unit.assertEqual(str(case.expectid),code )
AssertionError: '10001' != '20206'
- 10001
+ 20206



    </pre>
    </div>
    </td>
</tr>

<tr id='ft2_17' class='none'>
    <td class='failCase'><div class='testcase'>test_audit_17</div></td>
    <td colspan='5' align='center'>
    <!--默认收起错误信息 -Findyou
    <button id='btn_ft2_17' type="button"  class="btn btn-danger btn-xs collapsed" data-toggle="collapse" data-target='#div_ft2_17'>失败</button>
    <div id='div_ft2_17' class="collapse">  -->
    <!-- 默认展开错误信息 -Findyou -->
    <button id='btn_ft2_17' type="button"  class="btn btn-danger btn-xs" data-toggle="collapse" data-target='#div_ft2_17'>失败</button>
    <div id='div_ft2_17' class="collapse in" align="left">
    <pre>
    
ft2_17: Traceback (most recent call last):
  File "C:\Python37\lib\site-packages\ddt.py", line 145, in wrapper
    return func(self, *args, **kwargs)
  File "D:\PycharmProjects\test2-24\cookies0409\UseExcel\testcases\test_audit.py", line 40, in test_audit
    raise e                                            #沒有寫這個只會顯示用例執行次數，寫之後控制台顯示有誤信息
  File "D:\PycharmProjects\test2-24\cookies0409\UseExcel\testcases\test_audit.py", line 35, in test_audit
    self.unit.assertEqual(str(case.expectid),code )
AssertionError: '10001' != '20206'
- 10001
+ 20206



    </pre>
    </div>
    </td>
</tr>

<tr id='ft2_18' class='none'>
    <td class='failCase'><div class='testcase'>test_audit_18</div></td>
    <td colspan='5' align='center'>
    <!--默认收起错误信息 -Findyou
    <button id='btn_ft2_18' type="button"  class="btn btn-danger btn-xs collapsed" data-toggle="collapse" data-target='#div_ft2_18'>失败</button>
    <div id='div_ft2_18' class="collapse">  -->
    <!-- 默认展开错误信息 -Findyou -->
    <button id='btn_ft2_18' type="button"  class="btn btn-danger btn-xs" data-toggle="collapse" data-target='#div_ft2_18'>失败</button>
    <div id='div_ft2_18' class="collapse in" align="left">
    <pre>
    
ft2_18: Traceback (most recent call last):
  File "C:\Python37\lib\site-packages\ddt.py", line 145, in wrapper
    return func(self, *args, **kwargs)
  File "D:\PycharmProjects\test2-24\cookies0409\UseExcel\testcases\test_audit.py", line 40, in test_audit
    raise e                                            #沒有寫這個只會顯示用例執行次數，寫之後控制台顯示有誤信息
  File "D:\PycharmProjects\test2-24\cookies0409\UseExcel\testcases\test_audit.py", line 35, in test_audit
    self.unit.assertEqual(str(case.expectid),code )
AssertionError: '20210' != '20206'
- 20210
?    -
+ 20206
?     +



    </pre>
    </div>
    </td>
</tr>

<tr id='ft2_19' class='none'>
    <td class='failCase'><div class='testcase'>test_audit_19</div></td>
    <td colspan='5' align='center'>
    <!--默认收起错误信息 -Findyou
    <button id='btn_ft2_19' type="button"  class="btn btn-danger btn-xs collapsed" data-toggle="collapse" data-target='#div_ft2_19'>失败</button>
    <div id='div_ft2_19' class="collapse">  -->
    <!-- 默认展开错误信息 -Findyou -->
    <button id='btn_ft2_19' type="button"  class="btn btn-danger btn-xs" data-toggle="collapse" data-target='#div_ft2_19'>失败</button>
    <div id='div_ft2_19' class="collapse in" align="left">
    <pre>
    
ft2_19: Traceback (most recent call last):
  File "C:\Python37\lib\site-packages\ddt.py", line 145, in wrapper
    return func(self, *args, **kwargs)
  File "D:\PycharmProjects\test2-24\cookies0409\UseExcel\testcases\test_audit.py", line 40, in test_audit
    raise e                                            #沒有寫這個只會顯示用例執行次數，寫之後控制台顯示有誤信息
  File "D:\PycharmProjects\test2-24\cookies0409\UseExcel\testcases\test_audit.py", line 35, in test_audit
    self.unit.assertEqual(str(case.expectid),code )
AssertionError: '20211' != '20206'
- 20211
+ 20206



    </pre>
    </div>
    </td>
</tr>

<tr id='ft2_20' class='none'>
    <td class='failCase'><div class='testcase'>test_audit_20</div></td>
    <td colspan='5' align='center'>
    <!--默认收起错误信息 -Findyou
    <button id='btn_ft2_20' type="button"  class="btn btn-danger btn-xs collapsed" data-toggle="collapse" data-target='#div_ft2_20'>失败</button>
    <div id='div_ft2_20' class="collapse">  -->
    <!-- 默认展开错误信息 -Findyou -->
    <button id='btn_ft2_20' type="button"  class="btn btn-danger btn-xs" data-toggle="collapse" data-target='#div_ft2_20'>失败</button>
    <div id='div_ft2_20' class="collapse in" align="left">
    <pre>
    
ft2_20: Traceback (most recent call last):
  File "C:\Python37\lib\site-packages\ddt.py", line 145, in wrapper
    return func(self, *args, **kwargs)
  File "D:\PycharmProjects\test2-24\cookies0409\UseExcel\testcases\test_audit.py", line 40, in test_audit
    raise e                                            #沒有寫這個只會顯示用例執行次數，寫之後控制台顯示有誤信息
  File "D:\PycharmProjects\test2-24\cookies0409\UseExcel\testcases\test_audit.py", line 35, in test_audit
    self.unit.assertEqual(str(case.expectid),code )
AssertionError: '20211' != '20206'
- 20211
+ 20206



    </pre>
    </div>
    </td>
</tr>

<tr id='ft2_21' class='none'>
    <td class='failCase'><div class='testcase'>test_audit_21</div></td>
    <td colspan='5' align='center'>
    <!--默认收起错误信息 -Findyou
    <button id='btn_ft2_21' type="button"  class="btn btn-danger btn-xs collapsed" data-toggle="collapse" data-target='#div_ft2_21'>失败</button>
    <div id='div_ft2_21' class="collapse">  -->
    <!-- 默认展开错误信息 -Findyou -->
    <button id='btn_ft2_21' type="button"  class="btn btn-danger btn-xs" data-toggle="collapse" data-target='#div_ft2_21'>失败</button>
    <div id='div_ft2_21' class="collapse in" align="left">
    <pre>
    
ft2_21: Traceback (most recent call last):
  File "C:\Python37\lib\site-packages\ddt.py", line 145, in wrapper
    return func(self, *args, **kwargs)
  File "D:\PycharmProjects\test2-24\cookies0409\UseExcel\testcases\test_audit.py", line 40, in test_audit
    raise e                                            #沒有寫這個只會顯示用例執行次數，寫之後控制台顯示有誤信息
  File "D:\PycharmProjects\test2-24\cookies0409\UseExcel\testcases\test_audit.py", line 35, in test_audit
    self.unit.assertEqual(str(case.expectid),code )
AssertionError: '20211' != '20206'
- 20211
+ 20206



    </pre>
    </div>
    </td>
</tr>

<tr id='ft2_22' class='none'>
    <td class='failCase'><div class='testcase'>test_audit_22</div></td>
    <td colspan='5' align='center'>
    <!--默认收起错误信息 -Findyou
    <button id='btn_ft2_22' type="button"  class="btn btn-danger btn-xs collapsed" data-toggle="collapse" data-target='#div_ft2_22'>失败</button>
    <div id='div_ft2_22' class="collapse">  -->
    <!-- 默认展开错误信息 -Findyou -->
    <button id='btn_ft2_22' type="button"  class="btn btn-danger btn-xs" data-toggle="collapse" data-target='#div_ft2_22'>失败</button>
    <div id='div_ft2_22' class="collapse in" align="left">
    <pre>
    
ft2_22: Traceback (most recent call last):
  File "C:\Python37\lib\site-packages\ddt.py", line 145, in wrapper
    return func(self, *args, **kwargs)
  File "D:\PycharmProjects\test2-24\cookies0409\UseExcel\testcases\test_audit.py", line 40, in test_audit
    raise e                                            #沒有寫這個只會顯示用例執行次數，寫之後控制台顯示有誤信息
  File "D:\PycharmProjects\test2-24\cookies0409\UseExcel\testcases\test_audit.py", line 35, in test_audit
    self.unit.assertEqual(str(case.expectid),code )
AssertionError: '20211' != '20206'
- 20211
+ 20206



    </pre>
    </div>
    </td>
</tr>

<tr id='ft2_23' class='none'>
    <td class='failCase'><div class='testcase'>test_audit_23</div></td>
    <td colspan='5' align='center'>
    <!--默认收起错误信息 -Findyou
    <button id='btn_ft2_23' type="button"  class="btn btn-danger btn-xs collapsed" data-toggle="collapse" data-target='#div_ft2_23'>失败</button>
    <div id='div_ft2_23' class="collapse">  -->
    <!-- 默认展开错误信息 -Findyou -->
    <button id='btn_ft2_23' type="button"  class="btn btn-danger btn-xs" data-toggle="collapse" data-target='#div_ft2_23'>失败</button>
    <div id='div_ft2_23' class="collapse in" align="left">
    <pre>
    
ft2_23: Traceback (most recent call last):
  File "C:\Python37\lib\site-packages\ddt.py", line 145, in wrapper
    return func(self, *args, **kwargs)
  File "D:\PycharmProjects\test2-24\cookies0409\UseExcel\testcases\test_audit.py", line 40, in test_audit
    raise e                                            #沒有寫這個只會顯示用例執行次數，寫之後控制台顯示有誤信息
  File "D:\PycharmProjects\test2-24\cookies0409\UseExcel\testcases\test_audit.py", line 35, in test_audit
    self.unit.assertEqual(str(case.expectid),code )
AssertionError: '10001' != '20206'
- 10001
+ 20206



    </pre>
    </div>
    </td>
</tr>

<tr id='ft2_24' class='none'>
    <td class='failCase'><div class='testcase'>test_audit_24</div></td>
    <td colspan='5' align='center'>
    <!--默认收起错误信息 -Findyou
    <button id='btn_ft2_24' type="button"  class="btn btn-danger btn-xs collapsed" data-toggle="collapse" data-target='#div_ft2_24'>失败</button>
    <div id='div_ft2_24' class="collapse">  -->
    <!-- 默认展开错误信息 -Findyou -->
    <button id='btn_ft2_24' type="button"  class="btn btn-danger btn-xs" data-toggle="collapse" data-target='#div_ft2_24'>失败</button>
    <div id='div_ft2_24' class="collapse in" align="left">
    <pre>
    
ft2_24: Traceback (most recent call last):
  File "C:\Python37\lib\site-packages\ddt.py", line 145, in wrapper
    return func(self, *args, **kwargs)
  File "D:\PycharmProjects\test2-24\cookies0409\UseExcel\testcases\test_audit.py", line 40, in test_audit
    raise e                                            #沒有寫這個只會顯示用例執行次數，寫之後控制台顯示有誤信息
  File "D:\PycharmProjects\test2-24\cookies0409\UseExcel\testcases\test_audit.py", line 35, in test_audit
    self.unit.assertEqual(str(case.expectid),code )
AssertionError: '10001' != '20206'
- 10001
+ 20206



    </pre>
    </div>
    </td>
</tr>

<tr id='ft2_25' class='none'>
    <td class='failCase'><div class='testcase'>test_audit_25</div></td>
    <td colspan='5' align='center'>
    <!--默认收起错误信息 -Findyou
    <button id='btn_ft2_25' type="button"  class="btn btn-danger btn-xs collapsed" data-toggle="collapse" data-target='#div_ft2_25'>失败</button>
    <div id='div_ft2_25' class="collapse">  -->
    <!-- 默认展开错误信息 -Findyou -->
    <button id='btn_ft2_25' type="button"  class="btn btn-danger btn-xs" data-toggle="collapse" data-target='#div_ft2_25'>失败</button>
    <div id='div_ft2_25' class="collapse in" align="left">
    <pre>
    
ft2_25: Traceback (most recent call last):
  File "C:\Python37\lib\site-packages\ddt.py", line 145, in wrapper
    return func(self, *args, **kwargs)
  File "D:\PycharmProjects\test2-24\cookies0409\UseExcel\testcases\test_audit.py", line 40, in test_audit
    raise e                                            #沒有寫這個只會顯示用例執行次數，寫之後控制台顯示有誤信息
  File "D:\PycharmProjects\test2-24\cookies0409\UseExcel\testcases\test_audit.py", line 35, in test_audit
    self.unit.assertEqual(str(case.expectid),code )
AssertionError: '20210' != '20206'
- 20210
?    -
+ 20206
?     +



    </pre>
    </div>
    </td>
</tr>

<tr id='ft2_26' class='none'>
    <td class='failCase'><div class='testcase'>test_audit_26</div></td>
    <td colspan='5' align='center'>
    <!--默认收起错误信息 -Findyou
    <button id='btn_ft2_26' type="button"  class="btn btn-danger btn-xs collapsed" data-toggle="collapse" data-target='#div_ft2_26'>失败</button>
    <div id='div_ft2_26' class="collapse">  -->
    <!-- 默认展开错误信息 -Findyou -->
    <button id='btn_ft2_26' type="button"  class="btn btn-danger btn-xs" data-toggle="collapse" data-target='#div_ft2_26'>失败</button>
    <div id='div_ft2_26' class="collapse in" align="left">
    <pre>
    
ft2_26: Traceback (most recent call last):
  File "C:\Python37\lib\site-packages\ddt.py", line 145, in wrapper
    return func(self, *args, **kwargs)
  File "D:\PycharmProjects\test2-24\cookies0409\UseExcel\testcases\test_audit.py", line 40, in test_audit
    raise e                                            #沒有寫這個只會顯示用例執行次數，寫之後控制台顯示有誤信息
  File "D:\PycharmProjects\test2-24\cookies0409\UseExcel\testcases\test_audit.py", line 35, in test_audit
    self.unit.assertEqual(str(case.expectid),code )
AssertionError: '20211' != '20206'
- 20211
+ 20206



    </pre>
    </div>
    </td>
</tr>

<tr id='ft2_27' class='none'>
    <td class='failCase'><div class='testcase'>test_audit_27</div></td>
    <td colspan='5' align='center'>
    <!--默认收起错误信息 -Findyou
    <button id='btn_ft2_27' type="button"  class="btn btn-danger btn-xs collapsed" data-toggle="collapse" data-target='#div_ft2_27'>失败</button>
    <div id='div_ft2_27' class="collapse">  -->
    <!-- 默认展开错误信息 -Findyou -->
    <button id='btn_ft2_27' type="button"  class="btn btn-danger btn-xs" data-toggle="collapse" data-target='#div_ft2_27'>失败</button>
    <div id='div_ft2_27' class="collapse in" align="left">
    <pre>
    
ft2_27: Traceback (most recent call last):
  File "C:\Python37\lib\site-packages\ddt.py", line 145, in wrapper
    return func(self, *args, **kwargs)
  File "D:\PycharmProjects\test2-24\cookies0409\UseExcel\testcases\test_audit.py", line 40, in test_audit
    raise e                                            #沒有寫這個只會顯示用例執行次數，寫之後控制台顯示有誤信息
  File "D:\PycharmProjects\test2-24\cookies0409\UseExcel\testcases\test_audit.py", line 35, in test_audit
    self.unit.assertEqual(str(case.expectid),code )
AssertionError: '20211' != '20206'
- 20211
+ 20206



    </pre>
    </div>
    </td>
</tr>

<tr id='ft2_28' class='none'>
    <td class='failCase'><div class='testcase'>test_audit_28</div></td>
    <td colspan='5' align='center'>
    <!--默认收起错误信息 -Findyou
    <button id='btn_ft2_28' type="button"  class="btn btn-danger btn-xs collapsed" data-toggle="collapse" data-target='#div_ft2_28'>失败</button>
    <div id='div_ft2_28' class="collapse">  -->
    <!-- 默认展开错误信息 -Findyou -->
    <button id='btn_ft2_28' type="button"  class="btn btn-danger btn-xs" data-toggle="collapse" data-target='#div_ft2_28'>失败</button>
    <div id='div_ft2_28' class="collapse in" align="left">
    <pre>
    
ft2_28: Traceback (most recent call last):
  File "C:\Python37\lib\site-packages\ddt.py", line 145, in wrapper
    return func(self, *args, **kwargs)
  File "D:\PycharmProjects\test2-24\cookies0409\UseExcel\testcases\test_audit.py", line 40, in test_audit
    raise e                                            #沒有寫這個只會顯示用例執行次數，寫之後控制台顯示有誤信息
  File "D:\PycharmProjects\test2-24\cookies0409\UseExcel\testcases\test_audit.py", line 35, in test_audit
    self.unit.assertEqual(str(case.expectid),code )
AssertionError: '10001' != '20206'
- 10001
+ 20206



    </pre>
    </div>
    </td>
</tr>

<tr id='ft2_29' class='none'>
    <td class='failCase'><div class='testcase'>test_audit_29</div></td>
    <td colspan='5' align='center'>
    <!--默认收起错误信息 -Findyou
    <button id='btn_ft2_29' type="button"  class="btn btn-danger btn-xs collapsed" data-toggle="collapse" data-target='#div_ft2_29'>失败</button>
    <div id='div_ft2_29' class="collapse">  -->
    <!-- 默认展开错误信息 -Findyou -->
    <button id='btn_ft2_29' type="button"  class="btn btn-danger btn-xs" data-toggle="collapse" data-target='#div_ft2_29'>失败</button>
    <div id='div_ft2_29' class="collapse in" align="left">
    <pre>
    
ft2_29: Traceback (most recent call last):
  File "C:\Python37\lib\site-packages\ddt.py", line 145, in wrapper
    return func(self, *args, **kwargs)
  File "D:\PycharmProjects\test2-24\cookies0409\UseExcel\testcases\test_audit.py", line 40, in test_audit
    raise e                                            #沒有寫這個只會顯示用例執行次數，寫之後控制台顯示有誤信息
  File "D:\PycharmProjects\test2-24\cookies0409\UseExcel\testcases\test_audit.py", line 35, in test_audit
    self.unit.assertEqual(str(case.expectid),code )
AssertionError: '20211' != '20206'
- 20211
+ 20206



    </pre>
    </div>
    </td>
</tr>

<tr id='ft2_30' class='none'>
    <td class='failCase'><div class='testcase'>test_audit_30</div></td>
    <td colspan='5' align='center'>
    <!--默认收起错误信息 -Findyou
    <button id='btn_ft2_30' type="button"  class="btn btn-danger btn-xs collapsed" data-toggle="collapse" data-target='#div_ft2_30'>失败</button>
    <div id='div_ft2_30' class="collapse">  -->
    <!-- 默认展开错误信息 -Findyou -->
    <button id='btn_ft2_30' type="button"  class="btn btn-danger btn-xs" data-toggle="collapse" data-target='#div_ft2_30'>失败</button>
    <div id='div_ft2_30' class="collapse in" align="left">
    <pre>
    
ft2_30: Traceback (most recent call last):
  File "C:\Python37\lib\site-packages\ddt.py", line 145, in wrapper
    return func(self, *args, **kwargs)
  File "D:\PycharmProjects\test2-24\cookies0409\UseExcel\testcases\test_audit.py", line 40, in test_audit
    raise e                                            #沒有寫這個只會顯示用例執行次數，寫之後控制台顯示有誤信息
  File "D:\PycharmProjects\test2-24\cookies0409\UseExcel\testcases\test_audit.py", line 35, in test_audit
    self.unit.assertEqual(str(case.expectid),code )
AssertionError: '20211' != '20206'
- 20211
+ 20206



    </pre>
    </div>
    </td>
</tr>

<tr id='ft2_31' class='none'>
    <td class='failCase'><div class='testcase'>test_audit_31</div></td>
    <td colspan='5' align='center'>
    <!--默认收起错误信息 -Findyou
    <button id='btn_ft2_31' type="button"  class="btn btn-danger btn-xs collapsed" data-toggle="collapse" data-target='#div_ft2_31'>失败</button>
    <div id='div_ft2_31' class="collapse">  -->
    <!-- 默认展开错误信息 -Findyou -->
    <button id='btn_ft2_31' type="button"  class="btn btn-danger btn-xs" data-toggle="collapse" data-target='#div_ft2_31'>失败</button>
    <div id='div_ft2_31' class="collapse in" align="left">
    <pre>
    
ft2_31: Traceback (most recent call last):
  File "C:\Python37\lib\site-packages\ddt.py", line 145, in wrapper
    return func(self, *args, **kwargs)
  File "D:\PycharmProjects\test2-24\cookies0409\UseExcel\testcases\test_audit.py", line 40, in test_audit
    raise e                                            #沒有寫這個只會顯示用例執行次數，寫之後控制台顯示有誤信息
  File "D:\PycharmProjects\test2-24\cookies0409\UseExcel\testcases\test_audit.py", line 35, in test_audit
    self.unit.assertEqual(str(case.expectid),code )
AssertionError: '20211' != '20206'
- 20211
+ 20206



    </pre>
    </div>
    </td>
</tr>

<tr id='ft2_32' class='none'>
    <td class='failCase'><div class='testcase'>test_audit_32</div></td>
    <td colspan='5' align='center'>
    <!--默认收起错误信息 -Findyou
    <button id='btn_ft2_32' type="button"  class="btn btn-danger btn-xs collapsed" data-toggle="collapse" data-target='#div_ft2_32'>失败</button>
    <div id='div_ft2_32' class="collapse">  -->
    <!-- 默认展开错误信息 -Findyou -->
    <button id='btn_ft2_32' type="button"  class="btn btn-danger btn-xs" data-toggle="collapse" data-target='#div_ft2_32'>失败</button>
    <div id='div_ft2_32' class="collapse in" align="left">
    <pre>
    
ft2_32: Traceback (most recent call last):
  File "C:\Python37\lib\site-packages\ddt.py", line 145, in wrapper
    return func(self, *args, **kwargs)
  File "D:\PycharmProjects\test2-24\cookies0409\UseExcel\testcases\test_audit.py", line 40, in test_audit
    raise e                                            #沒有寫這個只會顯示用例執行次數，寫之後控制台顯示有誤信息
  File "D:\PycharmProjects\test2-24\cookies0409\UseExcel\testcases\test_audit.py", line 35, in test_audit
    self.unit.assertEqual(str(case.expectid),code )
AssertionError: '20211' != '20206'
- 20211
+ 20206



    </pre>
    </div>
    </td>
</tr>

<tr id='ft2_33' class='none'>
    <td class='failCase'><div class='testcase'>test_audit_33</div></td>
    <td colspan='5' align='center'>
    <!--默认收起错误信息 -Findyou
    <button id='btn_ft2_33' type="button"  class="btn btn-danger btn-xs collapsed" data-toggle="collapse" data-target='#div_ft2_33'>失败</button>
    <div id='div_ft2_33' class="collapse">  -->
    <!-- 默认展开错误信息 -Findyou -->
    <button id='btn_ft2_33' type="button"  class="btn btn-danger btn-xs" data-toggle="collapse" data-target='#div_ft2_33'>失败</button>
    <div id='div_ft2_33' class="collapse in" align="left">
    <pre>
    
ft2_33: Traceback (most recent call last):
  File "C:\Python37\lib\site-packages\ddt.py", line 145, in wrapper
    return func(self, *args, **kwargs)
  File "D:\PycharmProjects\test2-24\cookies0409\UseExcel\testcases\test_audit.py", line 40, in test_audit
    raise e                                            #沒有寫這個只會顯示用例執行次數，寫之後控制台顯示有誤信息
  File "D:\PycharmProjects\test2-24\cookies0409\UseExcel\testcases\test_audit.py", line 35, in test_audit
    self.unit.assertEqual(str(case.expectid),code )
AssertionError: '20211' != '20206'
- 20211
+ 20206



    </pre>
    </div>
    </td>
</tr>

<tr id='ft2_34' class='none'>
    <td class='failCase'><div class='testcase'>test_audit_34</div></td>
    <td colspan='5' align='center'>
    <!--默认收起错误信息 -Findyou
    <button id='btn_ft2_34' type="button"  class="btn btn-danger btn-xs collapsed" data-toggle="collapse" data-target='#div_ft2_34'>失败</button>
    <div id='div_ft2_34' class="collapse">  -->
    <!-- 默认展开错误信息 -Findyou -->
    <button id='btn_ft2_34' type="button"  class="btn btn-danger btn-xs" data-toggle="collapse" data-target='#div_ft2_34'>失败</button>
    <div id='div_ft2_34' class="collapse in" align="left">
    <pre>
    
ft2_34: Traceback (most recent call last):
  File "C:\Python37\lib\site-packages\ddt.py", line 145, in wrapper
    return func(self, *args, **kwargs)
  File "D:\PycharmProjects\test2-24\cookies0409\UseExcel\testcases\test_audit.py", line 40, in test_audit
    raise e                                            #沒有寫這個只會顯示用例執行次數，寫之後控制台顯示有誤信息
  File "D:\PycharmProjects\test2-24\cookies0409\UseExcel\testcases\test_audit.py", line 35, in test_audit
    self.unit.assertEqual(str(case.expectid),code )
AssertionError: '10001' != '20206'
- 10001
+ 20206



    </pre>
    </div>
    </td>
</tr>

<tr id='ft2_35' class='none'>
    <td class='failCase'><div class='testcase'>test_audit_35</div></td>
    <td colspan='5' align='center'>
    <!--默认收起错误信息 -Findyou
    <button id='btn_ft2_35' type="button"  class="btn btn-danger btn-xs collapsed" data-toggle="collapse" data-target='#div_ft2_35'>失败</button>
    <div id='div_ft2_35' class="collapse">  -->
    <!-- 默认展开错误信息 -Findyou -->
    <button id='btn_ft2_35' type="button"  class="btn btn-danger btn-xs" data-toggle="collapse" data-target='#div_ft2_35'>失败</button>
    <div id='div_ft2_35' class="collapse in" align="left">
    <pre>
    
ft2_35: Traceback (most recent call last):
  File "C:\Python37\lib\site-packages\ddt.py", line 145, in wrapper
    return func(self, *args, **kwargs)
  File "D:\PycharmProjects\test2-24\cookies0409\UseExcel\testcases\test_audit.py", line 40, in test_audit
    raise e                                            #沒有寫這個只會顯示用例執行次數，寫之後控制台顯示有誤信息
  File "D:\PycharmProjects\test2-24\cookies0409\UseExcel\testcases\test_audit.py", line 35, in test_audit
    self.unit.assertEqual(str(case.expectid),code )
AssertionError: '20211' != '20206'
- 20211
+ 20206



    </pre>
    </div>
    </td>
</tr>

<tr id='ft2_36' class='none'>
    <td class='failCase'><div class='testcase'>test_audit_36</div></td>
    <td colspan='5' align='center'>
    <!--默认收起错误信息 -Findyou
    <button id='btn_ft2_36' type="button"  class="btn btn-danger btn-xs collapsed" data-toggle="collapse" data-target='#div_ft2_36'>失败</button>
    <div id='div_ft2_36' class="collapse">  -->
    <!-- 默认展开错误信息 -Findyou -->
    <button id='btn_ft2_36' type="button"  class="btn btn-danger btn-xs" data-toggle="collapse" data-target='#div_ft2_36'>失败</button>
    <div id='div_ft2_36' class="collapse in" align="left">
    <pre>
    
ft2_36: Traceback (most recent call last):
  File "C:\Python37\lib\site-packages\ddt.py", line 145, in wrapper
    return func(self, *args, **kwargs)
  File "D:\PycharmProjects\test2-24\cookies0409\UseExcel\testcases\test_audit.py", line 40, in test_audit
    raise e                                            #沒有寫這個只會顯示用例執行次數，寫之後控制台顯示有誤信息
  File "D:\PycharmProjects\test2-24\cookies0409\UseExcel\testcases\test_audit.py", line 35, in test_audit
    self.unit.assertEqual(str(case.expectid),code )
AssertionError: '20211' != '20206'
- 20211
+ 20206



    </pre>
    </div>
    </td>
</tr>

<tr class='errorClass warning'>
    <td>test_invest.TestInvest: 測試投資</td>
    <td class="text-center">5</td>
    <td class="text-center">1</td>
    <td class="text-center">2</td>
    <td class="text-center">2</td>
    <td class="text-center"><a href="javascript:showClassDetail('c3',5)" class="detail" id='c3'>详细</a></td>
</tr>

<tr id='pt3_1' class='hiddenRow'>
    <td class='passCase'><div class='testcase'>test_invest_1</div></td>
    <td colspan='5' align='center'><span class="label label-success success">通过</span></td>
</tr>

<tr id='ft3_2' class='none'>
    <td class='errorCase'><div class='testcase'>test_invest_2</div></td>
    <td colspan='5' align='center'>
    <!--默认收起错误信息 -Findyou
    <button id='btn_ft3_2' type="button"  class="btn btn-danger btn-xs collapsed" data-toggle="collapse" data-target='#div_ft3_2'>错误</button>
    <div id='div_ft3_2' class="collapse">  -->
    <!-- 默认展开错误信息 -Findyou -->
    <button id='btn_ft3_2' type="button"  class="btn btn-danger btn-xs" data-toggle="collapse" data-target='#div_ft3_2'>错误</button>
    <div id='div_ft3_2' class="collapse in" align="left">
    <pre>
    
ft3_2: Traceback (most recent call last):
  File "C:\Python37\lib\site-packages\ddt.py", line 145, in wrapper
    return func(self, *args, **kwargs)
  File "D:\PycharmProjects\test2-24\cookies0409\UseExcel\testcases\test_invest.py", line 57, in test_invest
    case.data = replace(case.data)  # 若存在寫在配置文件中的賬號和密碼、項目名，就必須通過replace函數去拿到正則表達式匹配的值
  File "D:\PycharmProjects\test2-24\cookies0409\UseExcel\common\context.py", line 19, in replace
    da = str(da['id'])                   #從數據庫取出來的id是int
TypeError: 'NoneType' object is not subscriptable


    </pre>
    </div>
    </td>
</tr>

<tr id='ft3_3' class='none'>
    <td class='failCase'><div class='testcase'>test_invest_3</div></td>
    <td colspan='5' align='center'>
    <!--默认收起错误信息 -Findyou
    <button id='btn_ft3_3' type="button"  class="btn btn-danger btn-xs collapsed" data-toggle="collapse" data-target='#div_ft3_3'>失败</button>
    <div id='div_ft3_3' class="collapse">  -->
    <!-- 默认展开错误信息 -Findyou -->
    <button id='btn_ft3_3' type="button"  class="btn btn-danger btn-xs" data-toggle="collapse" data-target='#div_ft3_3'>失败</button>
    <div id='div_ft3_3' class="collapse in" align="left">
    <pre>
    
ft3_3: Traceback (most recent call last):
  File "C:\Python37\lib\site-packages\ddt.py", line 145, in wrapper
    return func(self, *args, **kwargs)
  File "D:\PycharmProjects\test2-24\cookies0409\UseExcel\testcases\test_invest.py", line 88, in test_invest
    raise e                                            #沒有寫這個只會顯示用例執行次數，寫之後控制台顯示有誤信息
  File "D:\PycharmProjects\test2-24\cookies0409\UseExcel\testcases\test_invest.py", line 67, in test_invest
    self.unit.assertEqual(str(case.expectid),code )
AssertionError: '10001' != '20212'
- 10001
+ 20212



    </pre>
    </div>
    </td>
</tr>

<tr id='ft3_4' class='none'>
    <td class='failCase'><div class='testcase'>test_invest_4</div></td>
    <td colspan='5' align='center'>
    <!--默认收起错误信息 -Findyou
    <button id='btn_ft3_4' type="button"  class="btn btn-danger btn-xs collapsed" data-toggle="collapse" data-target='#div_ft3_4'>失败</button>
    <div id='div_ft3_4' class="collapse">  -->
    <!-- 默认展开错误信息 -Findyou -->
    <button id='btn_ft3_4' type="button"  class="btn btn-danger btn-xs" data-toggle="collapse" data-target='#div_ft3_4'>失败</button>
    <div id='div_ft3_4' class="collapse in" align="left">
    <pre>
    
ft3_4: Traceback (most recent call last):
  File "C:\Python37\lib\site-packages\ddt.py", line 145, in wrapper
    return func(self, *args, **kwargs)
  File "D:\PycharmProjects\test2-24\cookies0409\UseExcel\testcases\test_invest.py", line 88, in test_invest
    raise e                                            #沒有寫這個只會顯示用例執行次數，寫之後控制台顯示有誤信息
  File "D:\PycharmProjects\test2-24\cookies0409\UseExcel\testcases\test_invest.py", line 67, in test_invest
    self.unit.assertEqual(str(case.expectid),code )
AssertionError: '10001' != '20111'
- 10001
+ 20111



    </pre>
    </div>
    </td>
</tr>

<tr id='ft3_5' class='none'>
    <td class='errorCase'><div class='testcase'>test_invest_5</div></td>
    <td colspan='5' align='center'>
    <!--默认收起错误信息 -Findyou
    <button id='btn_ft3_5' type="button"  class="btn btn-danger btn-xs collapsed" data-toggle="collapse" data-target='#div_ft3_5'>错误</button>
    <div id='div_ft3_5' class="collapse">  -->
    <!-- 默认展开错误信息 -Findyou -->
    <button id='btn_ft3_5' type="button"  class="btn btn-danger btn-xs" data-toggle="collapse" data-target='#div_ft3_5'>错误</button>
    <div id='div_ft3_5' class="collapse in" align="left">
    <pre>
    
ft3_5: Traceback (most recent call last):
  File "C:\Python37\lib\site-packages\ddt.py", line 145, in wrapper
    return func(self, *args, **kwargs)
  File "D:\PycharmProjects\test2-24\cookies0409\UseExcel\testcases\test_invest.py", line 37, in test_invest
    before_amount = money_be['leaveamount']
TypeError: 'NoneType' object is not subscriptable


    </pre>
    </div>
    </td>
</tr>

<tr class='passClass warning'>
    <td>test_login.Test_Login: 測試登錄接口</td>
    <td class="text-center">5</td>
    <td class="text-center">5</td>
    <td class="text-center">0</td>
    <td class="text-center">0</td>
    <td class="text-center"><a href="javascript:showClassDetail('c4',5)" class="detail" id='c4'>详细</a></td>
</tr>

<tr id='pt4_1' class='hiddenRow'>
    <td class='passCase'><div class='testcase'>test_login_1</div></td>
    <td colspan='5' align='center'><span class="label label-success success">通过</span></td>
</tr>

<tr id='pt4_2' class='hiddenRow'>
    <td class='passCase'><div class='testcase'>test_login_2</div></td>
    <td colspan='5' align='center'><span class="label label-success success">通过</span></td>
</tr>

<tr id='pt4_3' class='hiddenRow'>
    <td class='passCase'><div class='testcase'>test_login_3</div></td>
    <td colspan='5' align='center'><span class="label label-success success">通过</span></td>
</tr>

<tr id='pt4_4' class='hiddenRow'>
    <td class='passCase'><div class='testcase'>test_login_4</div></td>
    <td colspan='5' align='center'><span class="label label-success success">通过</span></td>
</tr>

<tr id='pt4_5' class='hiddenRow'>
    <td class='passCase'><div class='testcase'>test_login_5</div></td>
    <td colspan='5' align='center'><span class="label label-success success">通过</span></td>
</tr>

<tr class='errorClass warning'>
    <td>test_recharge.TestRecharge: 測試充值接口</td>
    <td class="text-center">2</td>
    <td class="text-center">0</td>
    <td class="text-center">0</td>
    <td class="text-center">2</td>
    <td class="text-center"><a href="javascript:showClassDetail('c5',2)" class="detail" id='c5'>详细</a></td>
</tr>

<tr id='ft5_1' class='none'>
    <td class='errorCase'><div class='testcase'>test_recharge_1</div></td>
    <td colspan='5' align='center'>
    <!--默认收起错误信息 -Findyou
    <button id='btn_ft5_1' type="button"  class="btn btn-danger btn-xs collapsed" data-toggle="collapse" data-target='#div_ft5_1'>错误</button>
    <div id='div_ft5_1' class="collapse">  -->
    <!-- 默认展开错误信息 -Findyou -->
    <button id='btn_ft5_1' type="button"  class="btn btn-danger btn-xs" data-toggle="collapse" data-target='#div_ft5_1'>错误</button>
    <div id='div_ft5_1' class="collapse in" align="left">
    <pre>
    
ft5_1: Traceback (most recent call last):
  File "C:\Python37\lib\site-packages\ddt.py", line 145, in wrapper
    return func(self, *args, **kwargs)
  File "D:\PycharmProjects\test2-24\cookies0409\UseExcel\testcases\test_recharge.py", line 31, in test_recharge
    before_amount = money_be['leaveamount']
TypeError: 'NoneType' object is not subscriptable


    </pre>
    </div>
    </td>
</tr>

<tr id='ft5_2' class='none'>
    <td class='errorCase'><div class='testcase'>test_recharge_2</div></td>
    <td colspan='5' align='center'>
    <!--默认收起错误信息 -Findyou
    <button id='btn_ft5_2' type="button"  class="btn btn-danger btn-xs collapsed" data-toggle="collapse" data-target='#div_ft5_2'>错误</button>
    <div id='div_ft5_2' class="collapse">  -->
    <!-- 默认展开错误信息 -Findyou -->
    <button id='btn_ft5_2' type="button"  class="btn btn-danger btn-xs" data-toggle="collapse" data-target='#div_ft5_2'>错误</button>
    <div id='div_ft5_2' class="collapse in" align="left">
    <pre>
    
ft5_2: Traceback (most recent call last):
  File "C:\Python37\lib\site-packages\ddt.py", line 145, in wrapper
    return func(self, *args, **kwargs)
  File "D:\PycharmProjects\test2-24\cookies0409\UseExcel\testcases\test_recharge.py", line 31, in test_recharge
    before_amount = money_be['leaveamount']
TypeError: 'NoneType' object is not subscriptable


    </pre>
    </div>
    </td>
</tr>

<tr class='errorClass warning'>
    <td>test_register.TestRegister: 測試註冊接口</td>
    <td class="text-center">7</td>
    <td class="text-center">0</td>
    <td class="text-center">5</td>
    <td class="text-center">2</td>
    <td class="text-center"><a href="javascript:showClassDetail('c6',7)" class="detail" id='c6'>详细</a></td>
</tr>

<tr id='ft6_1' class='none'>
    <td class='failCase'><div class='testcase'>test_register_1</div></td>
    <td colspan='5' align='center'>
    <!--默认收起错误信息 -Findyou
    <button id='btn_ft6_1' type="button"  class="btn btn-danger btn-xs collapsed" data-toggle="collapse" data-target='#div_ft6_1'>失败</button>
    <div id='div_ft6_1' class="collapse">  -->
    <!-- 默认展开错误信息 -Findyou -->
    <button id='btn_ft6_1' type="button"  class="btn btn-danger btn-xs" data-toggle="collapse" data-target='#div_ft6_1'>失败</button>
    <div id='div_ft6_1' class="collapse in" align="left">
    <pre>
    
ft6_1: Traceback (most recent call last):
  File "C:\Python37\lib\site-packages\ddt.py", line 145, in wrapper
    return func(self, *args, **kwargs)
  File "D:\PycharmProjects\test2-24\cookies0409\UseExcel\testcases\test_register.py", line 44, in test_register
    raise e                  #沒有寫這個只會顯示用例執行次數，寫之後控制台顯示有誤信息
  File "D:\PycharmProjects\test2-24\cookies0409\UseExcel\testcases\test_register.py", line 39, in test_register
    self.unit.assertEqual(case.expectid,resp.text)
AssertionError: '{"status":1,"code":"10001","data":null,"msg":"注册成功"}' != '{"status":0,"code":"20110","data":null,"msg":"手机号码已被注册"}'
- {"status":1,"code":"10001","data":null,"msg":"注册成功"}
?           ^           ---                       --
+ {"status":0,"code":"20110","data":null,"msg":"手机号码已被注册"}
?           ^         +++                       ++++++



    </pre>
    </div>
    </td>
</tr>

<tr id='ft6_2' class='none'>
    <td class='failCase'><div class='testcase'>test_register_2</div></td>
    <td colspan='5' align='center'>
    <!--默认收起错误信息 -Findyou
    <button id='btn_ft6_2' type="button"  class="btn btn-danger btn-xs collapsed" data-toggle="collapse" data-target='#div_ft6_2'>失败</button>
    <div id='div_ft6_2' class="collapse">  -->
    <!-- 默认展开错误信息 -Findyou -->
    <button id='btn_ft6_2' type="button"  class="btn btn-danger btn-xs" data-toggle="collapse" data-target='#div_ft6_2'>失败</button>
    <div id='div_ft6_2' class="collapse in" align="left">
    <pre>
    
ft6_2: Traceback (most recent call last):
  File "C:\Python37\lib\site-packages\ddt.py", line 145, in wrapper
    return func(self, *args, **kwargs)
  File "D:\PycharmProjects\test2-24\cookies0409\UseExcel\testcases\test_register.py", line 44, in test_register
    raise e                  #沒有寫這個只會顯示用例執行次數，寫之後控制台顯示有誤信息
  File "D:\PycharmProjects\test2-24\cookies0409\UseExcel\testcases\test_register.py", line 39, in test_register
    self.unit.assertEqual(case.expectid,resp.text)
AssertionError: '{"status":1,"code":"20111","data":null,"msg":"账号错误"}' != '{"status":0,"code":"20109","data":null,"msg":"手机号码格式不正确"}'
- {"status":1,"code":"20111","data":null,"msg":"账号错误"}
?           ^            ^^                     ^ ^^
+ {"status":0,"code":"20109","data":null,"msg":"手机号码格式不正确"}
?           ^            ^^                     ^^ ^^^^^^



    </pre>
    </div>
    </td>
</tr>

<tr id='ft6_3' class='none'>
    <td class='failCase'><div class='testcase'>test_register_3</div></td>
    <td colspan='5' align='center'>
    <!--默认收起错误信息 -Findyou
    <button id='btn_ft6_3' type="button"  class="btn btn-danger btn-xs collapsed" data-toggle="collapse" data-target='#div_ft6_3'>失败</button>
    <div id='div_ft6_3' class="collapse">  -->
    <!-- 默认展开错误信息 -Findyou -->
    <button id='btn_ft6_3' type="button"  class="btn btn-danger btn-xs" data-toggle="collapse" data-target='#div_ft6_3'>失败</button>
    <div id='div_ft6_3' class="collapse in" align="left">
    <pre>
    
ft6_3: Traceback (most recent call last):
  File "C:\Python37\lib\site-packages\ddt.py", line 145, in wrapper
    return func(self, *args, **kwargs)
  File "D:\PycharmProjects\test2-24\cookies0409\UseExcel\testcases\test_register.py", line 44, in test_register
    raise e                  #沒有寫這個只會顯示用例執行次數，寫之後控制台顯示有誤信息
  File "D:\PycharmProjects\test2-24\cookies0409\UseExcel\testcases\test_register.py", line 39, in test_register
    self.unit.assertEqual(case.expectid,resp.text)
AssertionError: '{"status":1,"code":"20111","data":null,"msg":"密码错误"}' != '{"status":0,"code":"20108","data":null,"msg":"密码长度必须为6~18"}'
- {"status":1,"code":"20111","data":null,"msg":"密码错误"}
?           ^            ^^                       ^^
+ {"status":0,"code":"20108","data":null,"msg":"密码长度必须为6~18"}
?           ^            ^^                       ^^^^^^^^^



    </pre>
    </div>
    </td>
</tr>

<tr id='ft6_4' class='none'>
    <td class='failCase'><div class='testcase'>test_register_4</div></td>
    <td colspan='5' align='center'>
    <!--默认收起错误信息 -Findyou
    <button id='btn_ft6_4' type="button"  class="btn btn-danger btn-xs collapsed" data-toggle="collapse" data-target='#div_ft6_4'>失败</button>
    <div id='div_ft6_4' class="collapse">  -->
    <!-- 默认展开错误信息 -Findyou -->
    <button id='btn_ft6_4' type="button"  class="btn btn-danger btn-xs" data-toggle="collapse" data-target='#div_ft6_4'>失败</button>
    <div id='div_ft6_4' class="collapse in" align="left">
    <pre>
    
ft6_4: Traceback (most recent call last):
  File "C:\Python37\lib\site-packages\ddt.py", line 145, in wrapper
    return func(self, *args, **kwargs)
  File "D:\PycharmProjects\test2-24\cookies0409\UseExcel\testcases\test_register.py", line 44, in test_register
    raise e                  #沒有寫這個只會顯示用例執行次數，寫之後控制台顯示有誤信息
  File "D:\PycharmProjects\test2-24\cookies0409\UseExcel\testcases\test_register.py", line 39, in test_register
    self.unit.assertEqual(case.expectid,resp.text)
AssertionError: '{"status":1,"code":"20103","data":null,"msg":"账号或密码不能为空"}' != '{"status":0,"code":"20103","data":null,"msg":"手机号不能为空"}'
- {"status":1,"code":"20103","data":null,"msg":"账号或密码不能为空"}
?           ^                                   ^ ---
+ {"status":0,"code":"20103","data":null,"msg":"手机号不能为空"}
?           ^                                   ^^



    </pre>
    </div>
    </td>
</tr>

<tr id='ft6_5' class='none'>
    <td class='failCase'><div class='testcase'>test_register_5</div></td>
    <td colspan='5' align='center'>
    <!--默认收起错误信息 -Findyou
    <button id='btn_ft6_5' type="button"  class="btn btn-danger btn-xs collapsed" data-toggle="collapse" data-target='#div_ft6_5'>失败</button>
    <div id='div_ft6_5' class="collapse">  -->
    <!-- 默认展开错误信息 -Findyou -->
    <button id='btn_ft6_5' type="button"  class="btn btn-danger btn-xs" data-toggle="collapse" data-target='#div_ft6_5'>失败</button>
    <div id='div_ft6_5' class="collapse in" align="left">
    <pre>
    
ft6_5: Traceback (most recent call last):
  File "C:\Python37\lib\site-packages\ddt.py", line 145, in wrapper
    return func(self, *args, **kwargs)
  File "D:\PycharmProjects\test2-24\cookies0409\UseExcel\testcases\test_register.py", line 44, in test_register
    raise e                  #沒有寫這個只會顯示用例執行次數，寫之後控制台顯示有誤信息
  File "D:\PycharmProjects\test2-24\cookies0409\UseExcel\testcases\test_register.py", line 39, in test_register
    self.unit.assertEqual(case.expectid,resp.text)
AssertionError: '{"status":1,"code":"20103","data":null,"msg":"账号或密码不能为空"}' != '{"status":0,"code":"20103","data":null,"msg":"密码不能为空"}'
- {"status":1,"code":"20103","data":null,"msg":"账号或密码不能为空"}
?           ^                                   ---
+ {"status":0,"code":"20103","data":null,"msg":"密码不能为空"}
?           ^



    </pre>
    </div>
    </td>
</tr>

<tr id='ft6_6' class='none'>
    <td class='errorCase'><div class='testcase'>test_register_6</div></td>
    <td colspan='5' align='center'>
    <!--默认收起错误信息 -Findyou
    <button id='btn_ft6_6' type="button"  class="btn btn-danger btn-xs collapsed" data-toggle="collapse" data-target='#div_ft6_6'>错误</button>
    <div id='div_ft6_6' class="collapse">  -->
    <!-- 默认展开错误信息 -Findyou -->
    <button id='btn_ft6_6' type="button"  class="btn btn-danger btn-xs" data-toggle="collapse" data-target='#div_ft6_6'>错误</button>
    <div id='div_ft6_6' class="collapse in" align="left">
    <pre>
    
ft6_6: Traceback (most recent call last):
  File "C:\Python37\lib\site-packages\ddt.py", line 145, in wrapper
    return func(self, *args, **kwargs)
  File "D:\PycharmProjects\test2-24\cookies0409\UseExcel\testcases\test_register.py", line 36, in test_register
    resp =self.r.request(case.method,case.url,case.data)
  File "D:\PycharmProjects\test2-24\cookies0409\UseExcel\common\requestHttp.py", line 17, in request
    url= self.config + url  #拼接URL
TypeError: can only concatenate str (not "NoneType") to str


    </pre>
    </div>
    </td>
</tr>

<tr id='ft6_7' class='none'>
    <td class='errorCase'><div class='testcase'>test_register_7</div></td>
    <td colspan='5' align='center'>
    <!--默认收起错误信息 -Findyou
    <button id='btn_ft6_7' type="button"  class="btn btn-danger btn-xs collapsed" data-toggle="collapse" data-target='#div_ft6_7'>错误</button>
    <div id='div_ft6_7' class="collapse">  -->
    <!-- 默认展开错误信息 -Findyou -->
    <button id='btn_ft6_7' type="button"  class="btn btn-danger btn-xs" data-toggle="collapse" data-target='#div_ft6_7'>错误</button>
    <div id='div_ft6_7' class="collapse in" align="left">
    <pre>
    
ft6_7: Traceback (most recent call last):
  File "C:\Python37\lib\site-packages\ddt.py", line 145, in wrapper
    return func(self, *args, **kwargs)
  File "D:\PycharmProjects\test2-24\cookies0409\UseExcel\testcases\test_register.py", line 36, in test_register
    resp =self.r.request(case.method,case.url,case.data)
  File "D:\PycharmProjects\test2-24\cookies0409\UseExcel\common\requestHttp.py", line 17, in request
    url= self.config + url  #拼接URL
TypeError: can only concatenate str (not "NoneType") to str


    </pre>
    </div>
    </td>
</tr>

<tr id='total_row' class="text-center active">
    <td>总计</td>
    <td>98</td>
    <td>21</td>
    <td>33</td>
    <td>44</td>
    <td>通过率：21.43%</td>
</tr>
</table>

<div id='ending'>&nbsp;</div>
    <div style=" position:fixed;right:50px; bottom:30px; width:20px; height:20px;cursor:pointer">
    <a href="#"><span class="glyphicon glyphicon-eject" style = "font-size:30px;" aria-hidden="true">
    </span></a></div>
    
</body>
</html>
